handbook: as/eaa updated (#1114)
italia/eidas-it-wallet-docs: handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md

Reviewer: ```suggestion
- [Modulo da compilare](#modulo-da-compilare)
```

Reviewer: ```suggestion
- **Step 1 | Progettazione caratteristiche EAA**: l'Ente approfondisce le Specifiche Tecniche del Sistema IT-Wallet e definisce le caratteristiche dell'Attestato Elettronico di Attributi relativo al proprio dataset, in termini di modalità di scoperta e ottenimento dell’EAA, casi d’uso, data model,  modalità di gestione di errori, stati e assistenza.
```

Reviewer: ```suggestion

```

Reviewer: ```suggestion
Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.
```

Reviewer: ```suggestion
È altresì importante che l’ente mantenga il proprio e-service aggiornato all’ultima versione indicata dalle Specifiche Tecniche IT-Wallet, facendo riferimento al branch LTS e assicurandosi in particolare di implementare sempre l’ultima versione di patch disponibile.

```

Reviewer: ```suggestion
Il referente dell'ambito sistemistico e il referente dell'ambito applicativo, così come definito nella sezione `assistenza` del modulo, devono contribuire alla diagnosi congiunta delle segnalazioni ricevute da Fornitore di Attestati Elettronici di Attributi (IPZS, nel caso di EAA di interesse pubblico) e Fornitori di Wallet (PagoPA, nel caso della soluzione pubblica IT-Wallet) e relativa risoluzione, secondo quanto definito dal [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet. 
```

Reviewer: ```suggestion
Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.
```

Reviewer: ```suggestion
## Modulo da compilare
```

Reviewer: ```suggestion
Il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) contiene le seguenti sezioni: 
```

Reviewer: ```suggestion
1. Prima di iniziare la compilazione, consulta i Template PDND Data Model e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche. 
```

@andreamisuraca i template sono disponibili? Se sì, a che link?

Reviewer: ```suggestion
1. Associa a ciascun dato che si intende rende disponibile all’interno dell’EAA un "nome campo", assicurandoti che sia parlante e che descriva adeguatamente il dato. 
```

Reviewer: ```suggestion
- Per l'errore 540 (EAA non esistente presso l'Authentic Source), utilizza il formato "esito": "causa", es.: "NOT_EXISTING": "l'EAA non è presente presso l'Authentic Source", "PENDING": "l'EAA è in attesa di emissione". 
```

Reviewer: ```suggestion
1. Mappa la condizione di applicabilità di ciascuno stato relativamente all'attestato in analisi;
```

Reviewer: ```suggestion
2. Descrivi l'azione necessaria per ripristinare lo stato di validità all’interno del campo "causa" (es. Chiedere la riemissione del documento presso uffici fisici / in digitale);  
```

Reviewer: ```suggestion
3. Definisci l’ ”azione utente” da condividere con l'utente (es. I tuoi dati sono stati aggiornati nella banca dati ANIS, scarica la nuova versione digitale del documento). Usa il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica. 
```

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 
```

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vedi [Appendice E](#appendice-e--assistenza).
```

Reviewer: ```suggestion
Nel definire ciascun dataset, i Titolari di Fonte Autentica non devono limitarsi a rendere disponibili le informazioni presenti nei propri sistemi, ma devono adottare una logica orientata al loro utilizzo. Ciò implica definire quali informazioni includere e come rappresentarle in funzione dello scopo dell’Attestato Elettronico, al fine di garantirne la comprensione e l’utilità per l’utente, nonché la verificabilità e l’interoperabilità tra sistemi.
```

Reviewer: ```suggestion
L’Ente deve quindi accuratamente definire, sulla base dei requisiti illustrati nella sezione [Focus sugli Attestati Elettronici di Attributi](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#focus-sugli-attestati-elettronici-di-attributi) delle Specifiche Tecniche, la struttura, la tipologia, la numerosità e l’ordinamento degli attributi che intende fornire all'interno di uno specifico dataset, nel rispetto dei seguenti principi: 
```

Reviewer: ```suggestion
- **Verificabilità**: garantire che i dati strutturati rappresentino in modo attendibile le caratteristiche e le qualità dell’utente, e che tali informazioni possano essere effettivamente verificate (*È possibile verificare la condizione o i diritti dell’utente attraverso uno o più attributi?*) 
```

Reviewer: ```suggestion
1. Per il codice 200 e per tutti gli errori obbligatori (400, 401, 404, 429, 500, 503) definisci la motivazione che ha scatenato l'errore popolando il campo "Causa"(es. Servizio momentaneamente non disponibile);
```

Reviewer: ```suggestion
2. Per ciascun errore descrivi l'azione necessaria per risolvere il problema popolando il campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa opzionalmente il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica;
```

Reviewer: ```suggestion
L’obiettivo della sezione "Assistenza" è quello di supportare gli Enti nella definizione dei contenuti per l'informazione e il supporto all'utente nell’interazione con l'EAA.  
```

Reviewer: ```suggestion
Destinatari: 

La sezione Assistenza si rivolge ad utenti amministrativi, responsabili di servizio o della comunicazione, operatori URP, gestori help desk o referenti del supporto tecnico dei fornitori IT. 
```

Reviewer: ```suggestion
  - [Modulo da compilare](#modulo-da-compilare)
           - [Denominazioni ufficiali](#denominazioni-ufficiali)
           - [Casi d'uso](#casi-duso)
           - [Data Model](#data-model)
           - [Mappatura errori](#mappatura-errori)
           - [Mappatura stati](#mappatura-stati)
           - [Assistenza](#assistenza)
```

le appendici diventano sottosezioni del capitolo Modulo da compilare

Reviewer: ```suggestion
### **Definire le denominazioni ufficiali **

Il Sistema IT-Wallet consente agli utenti di ottenere più EAA relativi a diversi Titolari Fonte Autentica. E’ quindi importante definire in maniera ufficiale e chiara per l’utente: 

- la denominazione con cui si intende veicolare il nome dell’Ente titolare; 
- la denominazione con cui si intende identificare in maniera univoca l’EAA relativo a ciascun dataset.  

Per approfondimenti vai alle Specifiche Tecniche: 
- [Focus sugli Attestati Elettronici di Attributi](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#focus-sugli-attestati-elettronici-di-attributi)

A tal fine, l’Ente deve compilare la sezione “Denominazioni ufficiali” del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vai all'apposita sezione del capitolo Modulo da compilare.
```

Reviewer: ```suggestion
In questa fase il Titolare di Fonte Autentica deve progettare il dataset dell'EAA di interesse.  

Ogni tipologia di EAA è infatti caratterizzata  da uno specifico dataset ed è identificata in maniera univoca dal valore `**datasetid**`, definito dalla Fonte Autentica. Grazie al `dataset_id` è possibile, quindi, individuare univocamente uno specifico dataset, utile soprattutto nei casi in cui, all’interno dello stesso e-service, coesistano più dataset distinti (es. e-service relativo all’ordine dei medici che raggruppa al suo interno più figure professionali: tesserino-chirurghi, tesserino-odontoiatri, etc.). 

Il parametro `datasetID` non va confuso con il parametro `**ObjectID**`, definito anch’esso dalla Fonte Autentica e che identifica univocamente ogni istanza di EAA, e che quindi consente di individuare l’istanza associata a uno specifico soggetto o utente. 
```

## Indice dei contenuti

- [Introduzione e contesto](#Introduzione-e-contesto)
- [Scopo e ambito di applicazione](#Scopo-e-ambito-di-applicazione)
- [Ruoli e responsabilità del Titolare di Fonte Autentica](#Ruoli-e-responsabilità-del-Titolare-di-Fonte-Autentica)
- [Come diventare Titolare di Fonte Autentica](#Come-diventare-Titolare-di-Fonte-Autentica)
  - [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa) 
  - [Step 2 | Pubblicazione in collaudo](#step-2--pubblicazione-in-collaudo) 
  - [Step 3 | Test in collaudo](#step-3--test-in-collaudo)
  - [Step 4 | Pubblicazione in produzione](#step-4--pubblicazione-in-produzione) 
  - [Step 5 | Test in produzione](#step-5--test-in-produzione)
  - [Step 6 | Pianificazione rilascio EAA](#step-6--pianificazione-rilascio-eaa)
  - [Step 7 | Manutenzione e assistenza](#step-7--manutenzione-e-assistenza)
- [Modulo da compilare](#modulo-da-compilare)
  - [Denominazioni ufficiali](#denominazioni-ufficiali)
  - [Casi d'uso](#casi-duso)
  - [Parametri di richiesta](#parametri-di-richiesta)

>   - [Data model](#data-model)

Reviewer: ```suggestion
  - [Dataset](#dataset)
```

*Figura 1: Ruolo del Titolare di Fonte Autentica nel Sistema IT-Wallet*

Di seguito il ruolo del Titolare di Fonte Autentica nel contesto del flusso di richiesta ed emissione di un EAA. 

![Flusso di richiesta ed emissione di un EAA nel Sistema IT-Wallet](images/figura-2-flusso-richiesta-emissione-eaa.svg)

*Figura 2: Flusso di richiesta ed emissione di un EAA nel Sistema IT-Wallet* 

> **NB:** Nel Sistema IT-Wallet, coerentemente con la normativa, IPZS (Istituto Poligrafico e Zecca dello Stato) è l'unico Fornitore di Attestati Elettronici di Attributi di Interesse Pubblico e PagoPA S.p.A. è l'unico fornitore di soluzione pubblica di IT-Wallet, ospitata all'interno dell'app IO, l'app dei servizi pubblici.

## Come diventare Titolare di Fonte Autentica

Per rivestire il ruolo di Titolare di Fonte Autentica, ciascun Ente interessato deve attenersi al seguente processo di onboarding tecnico, da considerarsi valido fino alla pubblicazione del Regolamento IT-Wallet e alla disponibilità di:

- Portale di Onboarding
- Registro delle Fonti Autentiche
- Catalogo degli Attestati Elettronici

In particolare, il processo prevede i seguenti step: 

> - **Step 1 | Progettazione caratteristiche EAA**: l'Ente approfondisce le Specifiche Tecniche del Sistema IT-Wallet e definisce le caratteristiche dell'Attestato Elettronico di Attributi relativo al proprio dataset, in termini di modalità di scoperta e ottenimento dell’EAA, casi d’uso, parametri di richiesta, dati di risposta, modalità di gestione di errori, stati e assistenza.

Reviewer: ```suggestion
- **Step 1 | Progettazione caratteristiche EAA**: l'Ente approfondisce le Specifiche Tecniche del Sistema IT-Wallet e definisce le caratteristiche dell'Attestato Elettronico di Attributi relativo al proprio dataset, in termini di modalità di scoperta e ottenimento dell’EAA, casi d’uso, dataset, modalità di gestione di errori, stati e assistenza.
```

### **Approfondire le Specifiche Tecniche del Sistema IT-Wallet**

La versione corrente delle Specifiche Tecniche è disponibile a [questo link](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/). Si raccomanda l'approfondimento, in particolare, delle sezioni: 

- [Design dell'Esperienza Utente](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html) 
- [Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)  
- [Endpoint delle Fonti Autentiche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-source-endpoint.html) 
- [Algoritmi Crittografici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/algorithms.html) 
- [e-Service PDND](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd.html) 
- [Soluzione del Fornitore di Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-issuer-solution.html)
- [Gestione degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/digital-credential-management.html)

> ### **Definire le denominazioni ufficiali **

Reviewer: ```suggestion
### **Definire le denominazioni ufficiali**
```

 Il parametro `dataset_id `non va confuso con il parametro `object_id`, definito anch’esso dalla Fonte Autentica e che identifica univocamente ogni istanza di EAA, e che quindi consente di individuare l’istanza associata a uno specifico soggetto o utente. 

Per poter richiamare l’e-service pubblicato relativo al dataset, dovrà essere utilizzato il Codice Fiscale come identificativo univoco dell’utente, qualora necessario. 

A tal fine, l’Ente deve definire il `dataset_id` e compilare la relativa sezione “Parametri di richiesta” del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per istruzioni vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).

### **Definire i dati di risposta**

Il Sistema IT-Wallet consente all’utente di ottenere in formato digitale i propri documenti, titoli e certificati sotto forma di EAA. Gli EAA sono rilasciati dal Fornitore di Attestati Elettronici di Attributi sulla base dei dati forniti dalla Fonte Autentica tramite l'e-service.  
In questa fase il Titolare di Fonte Autentica deve progettare il dataset dell'EAA di interesse.  

> Nel definire ciascun dataset, i Titolari di Fonte Autentica non devono limitarsi a rendere disponibili le informazioni presenti nei propri sistemi, ma devono adottare una logica orientata al loro utilizzo. Ciò implica definire quali informazioni includere e come rappresentarle in funzione dello scopo dell’Attestato Elettronico, al fine di garantirne la comprensione e l’utilità per l’utente, nonché la verificabilità e l’interoperabilità tra sistemi.

Reviewer: ```suggestion
### **Definire il dataset**

Il Sistema IT-Wallet consente al Fornitore di Attestati Elettronici di Attributi di ottenere i dati necessari dal Titolare di Fonte Autentica per generare l'attestato (EAA) richiesto dall'utente, tramite la fruizione di un apposito e-service messo a disposizione dal Titolare stesso.  

Per una corretta implementazione dell'e-service da pubblicare su PDND (vedi [Step 2](https://github.com/italia/eid-wallet-it-docs/blob/h-as-eaa/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#step-2--pubblicazione-in-collaudo)), è necessario che il Titolare di Fonte Autentica abbia adeguatamente definito a monte il dataset a livello di: `dataset_id`, `paramentri di richiesta` e `dati di risposta`.

Ogni tipologia di EAA è infatti caratterizzata da uno specifico dataset ed è identificata in maniera univoca dal valore `dataset_id`, definito dal Titolare di Fonte Autentica. Grazie al `dataset_id` è possibile individuare univocamente uno specifico dataset. Questo èutile soprattutto nei casi in cui, all’interno dello stesso e-service, coesistano più dataset distinti (es. e-service relativo all’ordine dei medici che raggruppa al suo interno più figure professionali: tesserino-chirurghi, tesserino-odontoiatri, etc.). Il parametro `dataset_id` non va confuso con il parametro `object_id`, definito anch’esso dal Titolare di Fonte Autentica, che identifica univocamente ogni istanza di EAA, e che consente invece di individuare l’istanza associata a uno specifico soggetto o utente. 

I dataset vengono messi a disposizione dei Fornitori di Attestati Elettronici di Attributi per mezzo di un e-service che può essere richiamato attraverso specifici parametri di richiesta definiti dal Titolare di Fonte Autentica. Oltre al `dataset_id`, l’Ente deve quindi definire anche i parametri di richiesta di ciascun dataset. Qualora fosse necessario identificare l'utente, il Codice Fiscale (CF) rappresenta l'identificativo univoco da utilizzare per le chiamate all'e-service. 

Infine, per permettere ai Fornitori di Attestati Elettronici di generare l’EAA, i Titolari di Fonte Autentica devono altresì definire i dati di risposta, ovvero i campi caratterizzanti ciascun dataset.  

Nel definire i dati di risposta di ciascun dataset, i Titolari di Fonte Autentica devono utilizzare banche dati complete nei contenuti e negli attributi, garantendo la massima profondità storica possibile, e adottare una logica orientata all’utilizzo dell’EAA relativo. Ciò implica definire quali informazioni includere nel dataset e come rappresentarle in funzione dello scopo dell’Attestato Elettronico, al fine di garantirne la comprensione e l’utilità per l’utente, nonché la verificabilità e l’interoperabilità tra sistemi. 
```

- **Comprensibilità**: assicurare che le informazioni siano immediatamente comprensibili, attraverso un linguaggio semplice e una semplificazione di contenuti complessi (*Le informazioni complesse sono state semplificate? Gli attributi sono espressi con un linguaggio chiaro e comprensibile?*) 
- **Pertinenza**: selezionare e includere esclusivamente gli attributi essenziali e rilevanti rispetto allo scopo dell’EAA, evitando informazioni superflue o non significative (*Quali attributi sono realmente necessari? Quali risultano poco rilevanti, superflui o ridondanti?*)

A scopo esemplificativo, di seguito sono riportati due esempi di rappresentazione grafica dell’EAA nella Soluzione Wallet di PagoPA S.p.A. Le configurazioni illustrate differiscono in relazione alle modalità di gestione degli attributi di II livello. 

![Esempio lista di attributi con valore](images/esempio-lista-di-attributi-con-valore.svg)

> *Figura 3: Esempio di resa grafica della Vista di dettaglio di un’EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di attributi con valore*

Reviewer: ```suggestion
*Figura 3: Esempio di resa grafica della Vista di dettaglio di un EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di attributi con valore*
```

![Esempio lista di descrizioni](images/esempio-lista-di-descrizioni.svg)

> *Figura 4: Esempio di resa grafica della Vista di dettaglio di un’EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di descrizioni*

Reviewer: ```suggestion
*Figura 4: Esempio di resa grafica della Vista di dettaglio di un EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di descrizioni*
```

> A tal fine, l’Ente deve compilare la sezione "Dati di risposta" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) relativa a ciascun dataset specificando il DatasetID e riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, ordinamento, etc.). Per istruzioni vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).

Reviewer: ```suggestion
A tal fine, l’ente deve adeguatamente definire il `dataset_id`, i `paramentri di richiesta` e i `dati di risposta`, e compilare la sezione “Dataset” del Modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per istruzioni vai all'apposita sezione del capitolo [Modulo da compilare](https://github.com/italia/eid-wallet-it-docs/blob/h-as-eaa/handbooks/it/authentic-sources/manuale-titolare-fonte-autentica.md#modulo-da-compilare).
```

Un'adeguata definizione dei dati di risposta pone le basi per una corretta implementazione dell'e-service da pubblicare su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)) ma è altresì importante considerare e rispettare i seguenti requisiti tecnici: 

- **Identificativo utente**: Qualora fosse necessario identificare l'utente, il Codice Fiscale (CF) rappresenta l'identificativo univoco da utilizzare per le chiamate all'e-service; 

> - **Completezza base di dati**: Ogni e-service pubblicato su PDND dovrà esporre un set di dati completo nel contenuto e negli attributi. È ammessa la pubblicazione di basi dati parziali relative a periodi temporali limitati.

Reviewer: ```suggestion

```

- Validare il modulo compilato utilizzando la funzionalità “valida” in testa al modulo ed esportarlo in formato JSON.
A compilazione, validazione ed export conclusi, l’Ente deve procedere con la sottomissione del modulo secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

L’ente dovrà creare, rinominare, compilare e validare un modulo per ciascun dataset relativo a un EAA di interesse.

Il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) contiene le seguenti sezioni: 

- Denominazioni ufficiali
- Casi d’uso
- Parametri di richiesta

> - Dati di risposta

Reviewer: ```suggestion
- Dataset
```

- Mappatura errori
- Mappatura stati
- Assistenza

Di seguito le indicazioni di compilazione di ciascuna sezione del modulo. 
### Denominazioni ufficiali 
 
L’obiettivo della sezione “Denominazioni ufficiali” è quello di definire la denominazione ufficiale con cui si intende veicolare il nome dell’Ente titolare e definire la denominazione ufficiale dell’EAA relativa allo specifico dataset, secondo quanto definito nello Step 1. 

> Destinatari: 

Reviewer: ```suggestion
**Destinatari**
```

La sezione “Denominazioni ufficiali” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto. 
 

> Istruzioni di compilazione: 

Reviewer: ```suggestion
**Istruzioni di compilazione**
```

- Compila i campi `nome ente titolare `e `nome eaa` seguendo i suggerimenti di compilazione.  
- Il campo `versione` è il numero di versione del modulo (valore fisso, es. 1.1.0) e non va confuso con una data di compilazione.

### Casi d'uso

L’obiettivo della sezione "Casi d'uso" è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti.

> Destinatari:

Reviewer: ```suggestion
**Destinatari**
```

La sezione “Casi d’uso” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto.

> Istruzioni di compilazione:

Reviewer: ```suggestion
**Istruzioni di compilazione**
```

1. **Domande pertinenti**: nel caso l'EAA non si riferisca a un documento fisico o digitale già esistente (es. patente), non compilare le domande che iniziano con "In caso di documento già esistente..";
2. **Domande a risposta aperta o sì/no**: ogni voce ha sempre tre campi; nell'interfaccia del modulo l'ordine è `domanda` (testo fisso), `risposta` (da compilare: stringa, oppure `true`/`false`/`null` in JSON per sì/no o bozza) e `esempio` (testo guida con esempio di risposta). Il campo `esempio` non sostituisce la `risposta`.

---

> ### Parametri di richiesta

Reviewer: ```suggestion
### Dataset
```

Destinatari: 

La sezione “Parametri di richiesta” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. 
L’obiettivo della sezione “Parametri di richiesta” è quello di supportare gli Enti nella definizione dei parametri necessari al Fornitore di Attestati Elettronici di Attributi per l’identificazione di uno specifico dataset. 
Destinatari: 

La sezione “Parametri di richiesta” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. 
Istruzioni di compilazione:

Definisci il valore `dataset_id`, necessario per identificare in maniera univoca uno specifico dataset, utilizzando preferibilmente un identificativo di tipo UUID oppure, in alternativa, adottando una stringa breve, descrittiva e priva di spazi in formato kebab case per nomi composti (es. tesserino-medici). 

> 

Reviewer: ```suggestion

L’obiettivo della sezione “Dataset” è quello di supportare gli Enti nella definizione dei parametri necessari al Fornitore di Attestati Elettronici di Attributi per l’identificazione di uno specifico dataset e nella definizione dei dati di risposta ovvero degli specifici campi del dataset restituito dall'e-service (Attestato Elettronico di Attributi).  

**Destinatari**

La sezione “Dataset” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT.  

**Istruzioni di compilazione** 

1. Definisci il valore `dataset_id`, necessario per identificare in maniera univoca uno specifico dataset, utilizzando preferibilmente un identificativo di tipo UUID oppure, in alternativa, adottando una stringa breve, descrittiva e priva di spazi in formato kebab case per nomi composti (es. tesserino-medici); 
2. Definisci i `parametri di richiesta`, necessari per l’emissione dell’EAA relativo al dataset attraverso la fruizione dell’e-service; 
3. Definisci i `dati risposta`. In particolare: 
- Prima di iniziare la compilazione, consulta i Template PDND Data Model e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche; 
- Associa a ciascun dato che si intende rendere disponibile all’interno dell’EAA un `nome campo`, visibile all’utente finale, assicurandoti che sia parlante e che descriva adeguatamente il dato. Ricorda che i dati possono essere strutturati su un massimo di due livelli; 
- Ordina i campi in modo da facilitare la leggibilità da parte dell’utente finale: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato. 

```

Da aggiungere il link esterno ai template PDND

La sezione “Data model” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la definizione del tipo di dati e dell’ordinamento). 
Istruzioni di compilazione:

1. Prima di iniziare la compilazione, consulta i Template PDND Data Model e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche; 
2. Associa a ciascun dato che si intende rendere disponibile all’interno dell’EAA un "nome campo", visibile all’utente finale, assicurandoti che sia parlante e che descriva adeguatamente il dato; 
3. Ordina i campi in modo da facilitare la leggibilità da parte dell’utente finale: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato.

> ---

Reviewer: ```suggestion

```

### Mappatura errori

L’obiettivo della sezione "Mappatura errori" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.

> Destinatari: 

Reviewer: ```suggestion
**Destinatari**
```

> Destinatari: 

Reviewer: ```suggestion

```

La sezione “Mappatura errori” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la formulazione di contenuti informativi e messaggi). 

> La sezione “Mappatura errori” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la formulazione di contenuti informativi e messaggi). 

Reviewer: ```suggestion

```

> Istruzioni di compilazione:

Reviewer: ```suggestion
**Istruzioni di compilazione**
```

2. Per ciascun errore descrivi l'azione necessaria per risolvere il problema nel campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa opzionalmente il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica;
3. Se ritenuto utile, compila allo stesso modo gli errori non obbligatori (es. 540 e 541) e/o aggiungi eventuali ulteriori errori specifici.

Nel caso di compilazione degli errori opzionali:
- Per l'errore 540 (EAA non esistente presso l'Authentic Source), utilizza il formato "esito": "causa", es.: "NOT_EXISTING": "l'EAA non è presente presso l'Authentic Source", "PENDING": "l'EAA è in attesa di emissione". 
- Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato). 

> 

Reviewer: ```suggestion
---

```

### Mappatura stati

L’obiettivo della sezione "Mappatura stati" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.

> Destinatari: 

Reviewer: ```suggestion
**Destinatari**
```

> 

Reviewer: ```suggestion
**Istruzioni di compilazione**
```

**Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo i valori letterali `Valido`, `Non Valido`, `Sospeso`, `Scaduto`, `Da aggiornare` (nell'ordine fisso previsto dal manuale). Per compatibilità con file già esportati, lo schema accetta anche la variante `Non valido` (v minuscola) al posto di `Non Valido` per il secondo elemento dell'array. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).

Per ciascuno dei **cinque** stati, il modulo prevede una `descrizione` testuale fissa (vincolata dallo schema): personalizza i messaggi rivolti all'utente tramite `azione utente`, `note` e, se utile, il campo opzionale `messaggio` (testo informativo aggiuntivo in IT-Wallet).

> **Istruzioni di compilazione**

Reviewer: ```suggestion

```
spostato più in alto

Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).

Estratto strutturale (nomi campo come nel JSON del modulo; `messaggio` è opzionale):

```json
"mappatura stati": [
  {
    "stato": "Valido",
    "descrizione": "L'EAA è valido e può essere utilizzato",
    "applicabile": true,
    "azione utente": "",
    "note": ""
  },
  {
    "stato": "Non Valido",
    "descrizione": "L'EAA non è più valido e dunque non può essere più utilizzato",
    "applicabile": false,
    "azione utente": "L'utente deve scaricare nuovamente l'EAA in app",
    "note": ""
  },
  {
    "stato": "Sospeso",
    "descrizione": "L'EAA è sospeso e non può essere temporaneamente utilizzato",
    "applicabile": false,
    "azione utente": "",
    "note": ""
  },
  {
    "stato": "Scaduto",
    "descrizione": "L'EAA ha superato la data di scadenza amministrativa e richiede rinnovo o nuova emissione",
    "applicabile": false,
    "azione utente": "",
    "note": ""
  }

> ]

Reviewer: ```suggestion

```

> 

Reviewer: ```suggestion
---
```

- testi informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 
1) Poniti le seguenti domande (puoi fare riferimento alla sezione "Casi d'uso", già compilata):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
2) Formula un testo informativo rivolto all’utente a partire dai contenuti sopra raccolti, assicurandoti che sia chiaro, semplice, diretto e conciso (circa 300 - 450 caratteri, spazi inclusi).

---

> 

Reviewer: ```suggestion
### Assistenza

L’obiettivo della sezione "Assistenza" è quello di supportare gli Enti nella definizione dei contenuti per l'informazione e il supporto all'utente nell’interazione con l'EAA.  

Nello specifico, l'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili i recapiti necessari per la risoluzione di eventuali malfunzionamenti, i canali disponibili per la gestione dell'assistenza agli utenti e i contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi da esporre all’utente in IT-Wallet. 

**Destinatari**

La sezione “Assistenza” si rivolge ad utenti amministrativi, responsabili di servizio o della comunicazione, operatori URP, gestori help desk o referenti del supporto tecnico dei fornitori IT. 

**Istruzioni di compilazione**

- Referenti: fornisci e mantieni aggiornati, i dati di contatto (nome, cognome, email, telefono) di almeno 1 referente per l’assistenza agli utenti, 1 referente in ambito applicativo e 1 referente in ambito sistemistico che possano prontamente collaborare con il Fornitore di Attestati Elettronici di Attributi (IPZS per gli Attestati Elettronici di interesse pubblico) e con il Fornitore di Wallet (PagoPA per la soluzione IT-Wallet pubblica) per la risoluzione congiunta di segnalazioni degli utenti e/o malfunzionamenti tra i servizi. 
- Canali: fornisci almeno un canale di assistenza (es. e-mail assistenza, numero di telefono, etc.) di responsabilità dell’Ente che rappresenti, nel ruolo di Titolare di Fonte Autentica, a cui si possa indirizzare l’utente per quelle richieste di supporto e segnalazioni non gestibili all’interno della soluzione IT-Wallet.  
- FAQ: per ciascuna voce compila `risposta` (stringa, booleano o null); `domanda` e `esempio` sono testi guida preimpostati nello stesso ordine usato per i casi d'uso (`domanda`, poi `esempio` con esempio di risposta, poi `risposta` da valorizzare).
- testi informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 
1) Poniti le seguenti domande (puoi fare riferimento alla sezione "Casi d'uso", già compilata): A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
2) Formula un testo informativo rivolto all’utente a partire dai contenuti sopra raccolti, assicurandoti che sia chiaro, semplice, diretto e conciso (circa 300 - 450 caratteri, spazi inclusi).

---

```

testo re-inserito perchè pareva cancellato

  - [Step 3 | Test in collaudo](#step-3--test-in-collaudo)
  - [Step 4 | Pubblicazione in produzione](#step-4--pubblicazione-in-produzione) 
  - [Step 5 | Test in produzione](#step-5--test-in-produzione)
  - [Step 6 | Pianificazione rilascio EAA](#step-6--pianificazione-rilascio-eaa)
  - [Step 7 | Manutenzione e assistenza](#step-7--manutenzione-e-assistenza)
- [Modulo da compilare](#modulo-da-compilare)
- [Appendice A – Casi d'uso](#appendice-a--casi-duso)
- [Appendice B – Parametri di richiesta](#appendice-b--parametri-di-richiesta)
- [Appendice C – Dati di risposta](#appendice-c--dati-di-risposta)
- [Appendice D – Mappatura errori](#appendice-d--mappatura-errori)
- [Appendice E – Mappatura stati](#appendice-e--mappatura-stati)
- [Appendice F – Assistenza](#appendice-f--assistenza)
  - [Modulo da compilare](#modulo-da-compilare)
           - [Denominazioni ufficiali](#denominazioni-ufficiali)
           - [Casi d'uso](#casi-duso)
           - [Parametri di richiesta](#parametri-di-richiesta)
           - [Dati di risposta](#dati-di-risposta)
           - [Mappatura errori](#mappatura-errori)
           - [Mappatura stati](#mappatura-stati)
           - [Assistenza](#assistenza)

> 

Reviewer: ```suggestion
- [Modulo da compilare](#modulo-da-compilare)
  - [Denominazioni ufficiali](#denominazioni-ufficiali)
  - [Casi d'uso](#casi-duso)
  - [Parametri di richiesta](#parametri-di-richiesta)
  - [Data model](#data-model)
  - [Mappatura errori](#mappatura-errori)
  - [Mappatura stati](#mappatura-stati)
  - [Assistenza](#assistenza)

```

  - **attraverso uno dei touchpoint della Fonte Autentica** (es. sito web), soluzione indicata per EAA di interesse locale o rivolti a un pubblico specifico (es. certificati, prenotazioni, etc.). La Fonte Autentica può così **guidare l'utente** verso l'ottenimento dell'EAA tramite Credential Offer. Questo flusso può affiancarsi o sostituire il precedente a seconda del tipo di EAA. Per approfondimento vai alle Specifiche Tecniche:
    - [Ottenimento dal Touchpoint del Titolare della Fonte Autentica](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-touchpoint-della-fonte-autentica)
    - [Credential Offer](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-issuance-low-level.html#flusso-credential-offer)
- il Titolare di Fonte Autentica può rispondere alla richiesta di emissione (tramite e-service) in: 
  - **modalità sincrona**, che consente l'ottenimento immediato dell'EAA da parte dell'utente e si configura come l'opzione preferibile;
  - **modalità differita**, che consente all'utente di ottenere l'EAA non contestualmente al momento della richiesta e si configura come l'opzione non preferibile.

> È necessario quindi che l'Ente definisca a monte le modalità di ottenimento dell‘EAA reso disponibile grazie ai propri dati, sulla base di determinati parametri: a chi si rivolge l'EAA (a tutta la popolazione o solo a una nicchia di persone?); cosa deve fare l'utente per ottenere l'EAA (è necessario essere in possesso di specifici prerequisiti? deve effettuare un processo di richiesta/adesione/pagamento? etc.); tramite quali canali l'utente potrà richiedere l'EAA e quando potrà ottenerlo (contestualmente o non contestualmente alla richiesta). 

Reviewer: ```suggestion
È necessario quindi che l'Ente definisca a monte le modalità di ottenimento dell‘EAA reso disponibile grazie ai propri dati, sulla base di determinati parametri: a chi si rivolge l'EAA (a tutta la popolazione o solo a una nicchia di persone?); cosa deve fare l'utente per ottenere l'EAA (è necessario essere in possesso di specifici prerequisiti? deve effettuare un processo di richiesta/adesione/pagamento? etc.); tramite quali canali l'utente potrà richiedere l'EAA e quando potrà ottenerlo (contestualmente o non contestualmente alla richiesta?).
```

  - [Presentazione in prossimità](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#presentazione-in-prossimita)
  - [Flusso di prossimità](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/proximity-flow.html)

La definizione dei casi d'uso da parte dell'Ente è fondamentale per: 

- **progettare un'esperienza d'uso che apporti valore reale** sia a cittadini che ai verificatori; è utile definire a monte quali potranno essere le occasioni d'uso dell'EAA prodotto con i propri dati, a partire dall'analisi dell'esperienza attuale di fruizione del corrispettivo documento fisico, se esistente (es. si pensi alla modalità di presentazione del codice a barre per l'uso della Tessera Sanitaria o del QR code per la verifica della Carta Europea della Disabilità); 
- **orientare il tipo di formato** con cui il Fornitore di Attestati Elettronici emetterà l'EAA (SD-JWT-VC per scenari in remoto e mdoc-CBOR per scenari in prossimità).

> A tal fine, l’Ente deve compilare la sezione "Casi d'uso" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vedi [Appendice A](#appendice-a--casi-duso).

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione "Casi d'uso" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).
```

### **Definire i parametri di richiesta**

> L'Ente deve definire i parametri che l'e-service Fonte Autentica riceve in ingresso per individuare l'utente o il contesto della richiesta (es. codice fiscale). A tal fine, compilare la sezione **Parametri di richiesta** del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per istruzioni vedi [Appendice B](#appendice-b--parametri-di-richiesta).

Reviewer: ```suggestion
Il Sistema IT-Wallet consente al Fornitore di Attestati Elettronici di Attributi di ottenere i dati dal Titolare di Fonte Autentica al fine di generare l’EAA di interesse per lo specifico utente che ne fa richiesta.  

Ogni tipologia di EAA è caratterizzata da uno specifico dataset ed è identificata in maniera univoca dal valore `dataset_id`, definito dalla Fonte Autentica. Grazie al ‘dataset_id’ è possibile, quindi, individuare univocamente uno specifico dataset, utile soprattutto nei casi in cui, all’interno dello stesso e-service, coesistano più dataset distinti (es. e-service relativo all’ordine dei medici che raggruppa al suo interno più figure professionali: tesserino-chirurghi, tesserino-odontoiatri, etc.). 
 Il parametro `dataset_id `non va confuso con il parametro `object_id`, definito anch’esso dalla Fonte Autentica e che identifica univocamente ogni istanza di EAA, e che quindi consente di individuare l’istanza associata a uno specifico soggetto o utente. 

Per poter richiamare l’e-service pubblicato relativo al dataset, dovrà essere utilizzato il Codice Fiscale come identificativo univoco dell’utente, qualora necessario. 

A tal fine, l’Ente deve definire il `dataset_id` e compilare la relativa sezione “Parametri di richiesta” del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per istruzioni vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).
```

### **Definire i dati di risposta**

Il Sistema IT-Wallet consente all’utente di ottenere in formato digitale i propri documenti, titoli e certificati sotto forma di EAA. Gli EAA sono rilasciati dal Fornitore di Attestati Elettronici di Attributi sulla base dei dati forniti dalla Fonte Autentica tramite l'e-service.  
In questa fase il Titolare di Fonte Autentica deve progettare il dataset dell'EAA di interesse.  

Ogni tipologia di EAA è infatti caratterizzata  da uno specifico dataset ed è identificata in maniera univoca dal valore `**datasetid**`, definito dalla Fonte Autentica. Grazie al `dataset_id` è possibile, quindi, individuare univocamente uno specifico dataset, utile soprattutto nei casi in cui, all’interno dello stesso e-service, coesistano più dataset distinti (es. e-service relativo all’ordine dei medici che raggruppa al suo interno più figure professionali: tesserino-chirurghi, tesserino-odontoiatri, etc.). 

> Il parametro `datasetID` non va confuso con il parametro `**ObjectID**`, definito anch’esso dalla Fonte Autentica e che identifica univocamente ogni istanza di EAA, e che quindi consente di individuare l’istanza associata a uno specifico soggetto o utente. 

Reviewer: ```suggestion

```

Spostato sopra

- **Pertinenza**: selezionare e includere esclusivamente gli attributi essenziali e rilevanti rispetto allo scopo dell’EAA, evitando informazioni superflue o non significative (*Quali attributi sono realmente necessari? Quali risultano poco rilevanti, superflui o ridondanti?*)

A scopo esemplificativo, di seguito sono riportati due esempi di rappresentazione grafica dell’EAA nella Soluzione Wallet di PagoPA S.p.A. Le configurazioni illustrate differiscono in relazione alle modalità di gestione degli attributi di II livello. 

![Esempio lista di attributi con valore](images/esempio-lista-di-attributi-con-valore.svg)

*Figura 3: Esempio di resa grafica della Vista di dettaglio di un’EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di attributi con valore*

![Esempio lista di descrizioni](images/esempio-lista-di-descrizioni.svg)

*Figura 4: Esempio di resa grafica della Vista di dettaglio di un’EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di descrizioni*

> A tal fine, l’Ente deve compilare la sezione **Dati di risposta** del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) relativa a ciascun dataset specificando il DatasetID e riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice C](#appendice-c--dati-di-risposta).

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione "Dati di risposta" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) relativa a ciascun dataset specificando il DatasetID e riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, ordinamento, etc.). Per istruzioni vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).
```

Gli stati ammissibili per un Attestato Elettronico di Attributi sono i seguenti: 

- **Valido** (valid): EAA emesso senza alcun segnale o nota di criticità verso l’utente; 
- **Sospeso** (suspended): EAA temporaneamente non valido, in uno stato di reversibilità. L'utente è invitato ad aspettare che lo stato torni ad essere valido (es. Patente ritirata); 
- **Non Valido** (invalid): EAA non più valido, in uno stato di irreversibilità. L'utente è invitato adeliminarlo oppure ad effettuare eventuali procedure propedeutiche ad avviare una nuova emissione che sovrascriva il precedente EAA (es. Patente riemessa);
 - **Da aggiornare** (attribute_update): EAA contenente una o più informazioni obsolete. L’utente è invitato ad aggiornare l’EAA attraverso un nuovo processo di emissione. 

> Oltre agli stati sopra elencati, è bene specificare che il ciclo di vita di un Attestato Elettronico sè influenzato anche dalla **scadenza tecnica** (definita dalla Fonte Autentica in sinergia con il Fornitore di Attestati Elettronici con l’obiettivo di innescare periodici aggiornamenti automatici dell’EAA per mitigare potenziali rischi di sicurezza) e dalla **scadenza amministrativa** (se resa disponibile tra gli attributi dell’EAA dal Titolare di Fonte Autentica). 

Reviewer: Copilot suggestion to be applied (typo)

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

> A tal fine, l'Ente deve compilare la sezione "Mappatura stati" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) per definire messaggi, testi informativi aggiuntivi (campo opzionale `messaggio`) e l'applicabilità dei quattro valori **`Valido`**, **`Non Valido`**, **`Sospeso`**, **`Scaduto`** e **`Da aggiornare`**.

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione "Mappatura stati" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) per definire messaggi, testi informativi aggiuntivi (campo opzionale `messaggio`) e l'applicabilità dei valori `Valido`, `Non Valido`, `Sospeso`, `Scaduto` e `Da aggiornare`.
```

## Step 7 | Manutenzione e assistenza

Una volta resi disponibili agli utenti gli EAA prodotti con i propri dati, la Fonte Autentica deve mantenere l'e-service e un ruolo attivo sia nella gestione dei dati (e del conseguente ciclo di vita degli EAA) che nell'assistenza agli utenti. 

Per i dettagli implementativi, consultare le Specifiche Tecniche, in particolare:

- [Assistenza utente](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente)
In questo step, l'Ente interessato deve:
### **Garantire la gestione e manutenzione dell'e-service**

L'Ente deve garantire il corretto funzionamento dell'e-service nel tempo, programmare adeguate azioni di monitoraggio e aggiornamento se richieste, ad esempio, da cambiamenti normativi o procedurali (es. nuovi dati, stati, casistiche di errore, etc.).

È altresì importante che l’ente mantenga il proprio e-service aggiornato all’ultima versione indicata dalle Specifiche Tecniche IT-Wallet, facendo riferimento al branch LTS e assicurandosi in particolare di implementare sempre l’ultima versione di patch disponibile.

### **Gestire problematiche e fornire assistenza agli utenti**

> L'Ente deve garantire un costante aggiornamento delle informazioni riportate delle informazioni riportate nella sezione "Assistenza" del modulo [Progettazione caratteristiche EAA(https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) al fine di:

Reviewer: suggestion to be applied

- **Garantire il supporto agli utenti** 
Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione "Assistenza" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.

## Modulo da compilare

Per assolvere a quanto previsto dallo Step 1, l‘Ente deve: 
- Accedere al modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa);
- Rinominare il modulo in "Caratteristiche EAA - [Nome EAA] - [Nome Ente Titolare]" (es. "Caratteristiche EAA – Titolo di studio - MIM");

> - Compilare il modulo in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di casi d'uso, dati, mappatura errori, stati e assistenza. E' possibile salvare localmente nel web browser un modulo in fase di compulazione ed è possibile l'import di JSON/CSV. Nel caso in cui più soggetti siano incaricati della compilazione del modulo, è quindi possibile esportare e condividere una bozza parzialmente compilata affinché possa essere successivamente caricata e completata da un altro soggetto. È inoltre possibile lavorare in parallelo su copie distinte del modulo; in tal caso, è necessario assicurarsi che tutti i contributi confluiscano in un’unica versione finale completa in ogni sua parte; 

Reviewer: suggestion to be applied

- Validare il modulo compilato utilizzando la funzionalità “valida” in testa al modulo ed esportarlo in formato JSON.
A compilazione, validazione ed export conclusi, l’Ente deve procedere con la sottomissione del modulo secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

L’ente dovrà creare, rinominare, compilare e validare un modulo per ciascun dataset relativo a un EAA di interesse.

Il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) contiene le seguenti sezioni: 

- Denominazioni ufficiali
- Casi d’uso
- Parametri di richiesta
- Dati di risposta
- Mappatura errori
- Mappatura stati
- Assistenza

> Di seguito le indicazioni di compilazione di ciascuna sezione del modulo. 

Reviewer: ```suggestion
- Assistenza

Di seguito le indicazioni di compilazione di ciascuna sezione del modulo. 
```

### Denominazioni ufficiali 
 
L’obiettivo della sezione “Denominazioni ufficiali” è quello di definire la denominazione ufficiale con cui si intende veicolare il nome dell’Ente titolare e definire la denominazione ufficiale dell’EAA relativa allo specifico dataset, secondo quanto definito nello Step 1. 

Destinatari: 

> La sezione “Denominazioni ufficiali” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto. 

Reviewer: ```suggestion
Destinatari: 

La sezione “Denominazioni ufficiali” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto. 
```

 
Istruzioni di compilazione: 
- Compila i campi `nome ente titolare `e `nome eaa` seguendo i suggerimenti di compilazione.  
- Il campo `versione` è il numero di versione del modulo (valore fisso, es. 1.1.0) e non va confuso con una data di compilazione.

### Parametri di richiesta

Definisci l'array **`parametri di richiesta`** con i parametri in ingresso all'e-service. Vedi [Appendice B](#appendice-b--parametri-di-richiesta).

### Dati di risposta

Definisci l'array **`dati di risposta`** con i campi restituiti dal dataset. Vedi [Appendice C](#appendice-c--dati-di-risposta).

> 

Reviewer: ```suggestion
```

sezioni già presenti sotto, correttamente posizionate dopo la sezione casi d'uso

### Casi d'uso

L’obiettivo della sezione "Casi d'uso" è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti.

Destinatari:
La sezione “Casi d’uso” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto.

Istruzioni di compilazione:

> 1. **domande pertinenti**: nel caso l'EAA non si riferisca a un documento fisico o digitale già esistente (es. patente), non compilare le domande che iniziano con "In caso di documento già esistente..";

Reviewer: ```suggestion
1. **Domande pertinenti**: nel caso l'EAA non si riferisca a un documento fisico o digitale già esistente (es. patente), non compilare le domande che iniziano con "In caso di documento già esistente..";
```

> 2. **Domande a risposta aperta o sì/no**: ogni voce ha sempre tre campi; nell'interfaccia del modulo l'ordine è **`domanda`** (testo fisso), **`risposta`** (da compilare: stringa, oppure `true`/`false`/`null` in JSON per sì/no o bozza) e **`esempio`** (testo guida con esempio di risposta). Il campo `esempio` non sostituisce la `risposta`.

Reviewer: ```suggestion
2. **Domande a risposta aperta o sì/no**: ogni voce ha sempre tre campi; nell'interfaccia del modulo l'ordine è `domanda` (testo fisso), `risposta` (da compilare: stringa, oppure `true`/`false`/`null` in JSON per sì/no o bozza) e `esempio` (testo guida con esempio di risposta). Il campo `esempio` non sostituisce la `risposta`.
```

> 3. **Nomi delle proprietà nel JSON**: usa le chiavi esattamente come nello schema di validazione (parole separate da **un solo spazio**; evita doppi spazi, ad esempio `canali richiesta doc preesistente`, `richieste fisico vs digitale doc preesistente`, `normativa doc preesistente`).

Reviewer: ```suggestion

```

Per istruzioni dettagliate vedi [Appendice A](#appendice-a--casi-duso).

**Esempio di compilazione** (fragmento):

```json
{
  "denominazioni ufficiali": {
    "nome ente titolare": "Ministero dell'Istruzione",
    "nome eaa": "Titolo di studio",
    "versione": "1.1.0"
  },
  "casi d'uso": {
    "target utenti": {
      "utenti doc preesistente": {
        "domanda": "Chi può ad oggi richiedere il documento? ...",
        "risposta": "Tutti i cittadini italiani in possesso di diploma rilasciato da istituto italiano",
        "esempio": "Solo maggiorenni residenti in una specifica regione / ..."
      }
    }
  }
}

> ```

Reviewer: ```suggestion

```

---

> ## Appendice B – Parametri di richiesta

Reviewer: ```suggestion
### Parametri di richiesta
```

Questa appendice descrive la sezione **`parametri di richiesta`** del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).

> **Obiettivo**

Reviewer: ```suggestion

```

superato. eliminato per coerenza con le altre sezioni

> Definire i parametri in ingresso all'endpoint e-service (es. codice fiscale, identificativi di pratica) necessari per recuperare i dati del dataset.

Reviewer: ```suggestion
L’obiettivo della sezione “Parametri di richiesta” è quello di supportare gli Enti nella definizione dei parametri necessari al Fornitore di Attestati Elettronici di Attributi per l’identificazione di uno specifico dataset. 
```

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Parametri di richiesta” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. 
```

> **Istruzioni di compilazione**

Reviewer: ```suggestion
Istruzioni di compilazione:
```

1. Per ciascun parametro indica `nome parametro`, `descrizione`, `nome campo` (etichetta per l'utente o per la documentazione), `tipologia`, `obbligatorio`, `lunghezza massima caratteri` e, se applicabile, `valori ammessi`.

> 2. Allinea i parametri di richiesta ai campi previsti dall'OpenAPI dell'e-service Fonte Autentica.

Reviewer: ```suggestion
Definisci il valore `dataset_id`, necessario per identificare in maniera univoca uno specifico dataset, utilizzando preferibilmente un identificativo di tipo UUID oppure, in alternativa, adottando una stringa breve, descrittiva e priva di spazi in formato kebab case per nomi composti (es. tesserino-medici). 
```

**Esempio** (array di primo livello nel JSON):

```json
"parametri di richiesta": [
  {
    "nome parametro": "tax_code",
    "descrizione": "codice fiscale dell'utente",
    "nome campo": "Codice Fiscale",
    "esempio campo compilato": "DLNRSL88L51C348G",
    "obbligatorio": true,
    "tipologia": "ALFANUMERICO",
    "lunghezza massima caratteri": 16,
    "valori ammessi": [],
    "note": ""
  }
]

> ```

Reviewer: ```suggestion

```

---

> ## Appendice C – Dati di risposta

Reviewer: ```suggestion
### Data model
```

Questa appendice descrive la sezione **`dati di risposta`** del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire.

> **Obiettivo**

Reviewer: ```suggestion

```

Eliminato in coerenza con le altre sezioni

L’obiettivo della sezione `dati di risposta` è supportare gli Enti nella definizione dei campi del dataset restituito dall'e-service (Attestato Elettronico di Attributi).

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Data model” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la definizione del tipo di dati e dell’ordinamento). 
```

> **Istruzioni di compilazione**

Reviewer: ```suggestion
Istruzioni di compilazione:
```

1. Associa a ciascun dato un `nome campo` parlante e coerente con l'uso in Wallet.
2. Ordina i campi per leggibilità: prima dati anagrafici, poi dati specifici dell'attestato.

> 3. (Opzionale) Compila **`lista attributi`** per documentare la resa in app (categoria, parametro, nome campo, descrizione).

Reviewer: ```suggestion
1. Prima di iniziare la compilazione, consulta i Template PDND Data Model e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche; 
2. Associa a ciascun dato che si intende rendere disponibile all’interno dell’EAA un "nome campo", visibile all’utente finale, assicurandoti che sia parlante e che descriva adeguatamente il dato; 
3. Ordina i campi in modo da facilitare la leggibilità da parte dell’utente finale: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato.
```

Link to "Template PDND Data Model" to be added

---

> ## Appendice D – Mappatura errori

Reviewer: ```suggestion
### Mappatura errori
```

Questa appendice descrive le istruzioni di compilazione della sezione `mappatura errori` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

> **Obiettivo**

Reviewer: ```suggestion

```

eliminato in coerenza alle altre sezioni

> L’obiettivo della sezione `mappatura errori` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.

Reviewer: ```suggestion
L’obiettivo della sezione "Mappatura errori" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.
```

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Mappatura errori” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la formulazione di contenuti informativi e messaggi). 
```

> **Istruzioni di compilazione**

Reviewer: ```suggestion
Istruzioni di compilazione:
```

> 1. Per il codice 200 e per tutti gli errori obbligatori (400, 401, 404, 429, 500, 503) definisci la motivazione che ha scatenato l'errore e popola il campo "Causa"(es. Servizio momentaneamente non disponibile);

Reviewer: ```suggestion
1. Per il codice 200 e per tutti gli errori obbligatori (400, 401, 404, 429, 500, 503) definisci la motivazione che ha scatenato l'errore popolando il campo "Causa"(es. Servizio momentaneamente non disponibile);
```

> 2. Per ciascun errore descrivi l'azione necessaria per risolvere il problema nel campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica;

Reviewer: ```suggestion
2. Per ciascun errore descrivi l'azione necessaria per risolvere il problema nel campo "Azione utente". (es. Ti invitiamo a riprovare più tardi). Usa opzionalmente il campo "Note" per aggiungere ulteriori informazioni utili o una spiegazione del perché proponiamo all'utente di compiere un'azione specifica;
```

"mappatura errori": [
    {
      "codice": 200,
      "esito": "Attestato digitale valido",
      "applicabile": true,
      "causa": "Vengono ritornati correttamente i dati, nessuna risposta di errore. Qualora non ci fossero azioni utente da eseguire, riportare stringa vuota.",
      "azione_utente": "",
      "note": ""
    },
    {
      "codice": 404,
      "esito": "Not found",
      "applicabile": true,
      "causa": "Non sono stati trovati documenti di titolarità dell'utente",
      "azione_utente": "Chiudere e riprovare successivamente",
      "note": "L'utente deve prima acquisire la titolarità del documento per ottenerne la versione digitale"
    },
    {
      "codice": 540,
      "esito": "EAA non esistente presso l'Authentic Source",
      "applicabile": false,
      "causa": "",
      "azione_utente": "",
      "note": ""
    },
    {
      "codice": 541,
      "esito": "EAA in stato non valido o sospeso",
      "applicabile": false,

> *Tabella 3 – Mappatura Errori (sezione `mappatura errori`)*

Reviewer: ```suggestion

```

> ## Appendice E – Mappatura stati

Reviewer: ```suggestion
### Mappatura stati
```

Questa appendice descrive le istruzioni di compilazione della sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

> **Obiettivo**

Reviewer: ```suggestion

```

> L’obiettivo della sezione `mappatura stati` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.

Reviewer: ```suggestion
L’obiettivo della sezione "Mappatura stati" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.
```

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Mappatura stati” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la definizione del ciclo vita e la relativa formulazione di contenuti informativi e messaggi). 
```

> **Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo i valori letterali **`Valido`**, **`Non Valido`**, **`Sospeso`**, **`Scaduto`**, **`Da aggiornare`** (nell'ordine fisso previsto dal template). Per compatibilità con file già esportati, lo schema accetta anche la variante **`Non valido`** (v minuscola) al posto di **Non Valido** per il secondo elemento dell'array. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).

Reviewer: ```suggestion
**Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo i valori letterali `Valido`, `Non Valido`, `Sospeso`, `Scaduto`, `Da aggiornare` (nell'ordine fisso previsto dal manuale). Per compatibilità con file già esportati, lo schema accetta anche la variante `Non valido` (v minuscola) al posto di `Non Valido` per il secondo elemento dell'array. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).
```

> Per ciascuno dei **cinque** stati, il modulo prevede una **descrizione** testuale fissa (vincolata dallo schema): personalizza i messaggi rivolti all'utente tramite **`azione utente`**, **`note`** e, se utile, il campo opzionale **`messaggio`** (testo informativo aggiuntivo in IT-Wallet).

Reviewer: ```suggestion
Per ciascuno dei **cinque** stati, il modulo prevede una `descrizione` testuale fissa (vincolata dallo schema): personalizza i messaggi rivolti all'utente tramite `azione utente`, `note` e, se utile, il campo opzionale `messaggio` (testo informativo aggiuntivo in IT-Wallet).
```

**Istruzioni di compilazione**

1. Mappa la condizione di applicabilità di ciascuno stato relativamente all'attestato in analisi (`applicabile`: `true` / `false`);

> 2. Il campo **descrizione** per ciascuno stato è quello predefinito del template (vincolato dallo schema e non modificabile); usa **`azione utente`**, **`note`** e opzionalmente **`messaggio`** per chiarire al cittadino cosa fare;

Reviewer: ```suggestion
2. Il campo `descrizione` per ciascuno stato è quello predefinito del modulo (vincolato dallo schema e non modificabile); usa `azione utente`, `note` e opzionalmente `messaggio` per chiarire al cittadino cosa fare;
```

> 3. Definisci l'**azione utente** da condividere con l'utente (es. «I tuoi dati sono stati aggiornati nella banca dati ANIS, scarica la nuova versione digitale del documento»). Usa il campo **note** per ulteriori indicazioni operative;

Reviewer: ```suggestion
3. Definisci la `causa` del cambiamento di stato e l'`azione utente` ovvero il messaggio da comunicare all’utente in caso di cambiamento stato (es. “I tuoi dati sono stati aggiornati nella banca dati ANIS, scarica la nuova versione digitale del documento”). Usa opzionalmente il campo `note` per aggiungere ulteriori informazioni utili o indicazioni operative. 
```

> 4. Se serve un messaggio dedicato in app oltre a descrizione e azione utente, valorizza opzionalmente **messaggio**; altrimenti ometti il campo o lascialo vuoto;

Reviewer: ```suggestion

```

> 5. Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).

Reviewer: ```suggestion

Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).
```

Estratto strutturale (nomi campo come nel JSON del modulo; `messaggio` è opzionale):

```json
"mappatura stati": [
  {
    "stato": "Valido",
    "descrizione": "L'EAA è valido e può essere utilizzato",
    "applicabile": true,
    "azione utente": "",
    "note": ""
  },
  {
    "stato": "Non Valido",
    "descrizione": "L'EAA non è più valido e dunque non può essere più utilizzato",
    "applicabile": false,
    "azione utente": "L'utente deve scaricare nuovamente l'EAA in app",
    "note": ""
  },
  {
    "stato": "Sospeso",
    "descrizione": "L'EAA è sospeso e non può essere temporaneamente utilizzato",
    "applicabile": false,
    "azione utente": "",
    "note": ""
  },
  {
    "stato": "Scaduto",
    "descrizione": "L'EAA ha superato la data di scadenza amministrativa e richiede rinnovo o nuova emissione",
    "applicabile": false,
    "azione utente": "",
    "note": ""
  }
]

> ```

Reviewer: ```suggestion

```

> ## Appendice F – Assistenza

Reviewer: ```suggestion
### Assistenza
```

Questa appendice descrive le istruzioni di compilazione della sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

> **Obiettivo** 

Reviewer: ```suggestion

```

> L’obiettivo della sezione `assistenza` è quello di supportare gli Enti nella definizione dei contenuti per l'informazione e il supporto all'utente nell’interazione con l'EAA.  

Reviewer: ```suggestion
L’obiettivo della sezione "Assistenza" è quello di supportare gli Enti nella definizione dei contenuti per l'informazione e il supporto all'utente nell’interazione con l'EAA.  
```

Nello specifico, l'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili i recapiti necessari per la risoluzione di eventuali malfunzionamenti, i canali disponibili per la gestione dell'assistenza agli utenti e i contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi da esporre all’utente in IT-Wallet. 

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Assistenza” si rivolge ad utenti amministrativi, responsabili di servizio o della comunicazione, operatori URP, gestori help desk o referenti del supporto tecnico dei fornitori IT. 
```

> **Istruzioni di compilazione** 

Reviewer: ```suggestion
Istruzioni di compilazione:
```

- Referenti: fornisci e mantieni aggiornati, i dati di contatto (nome, cognome, email, telefono) di almeno 1 referente per l’assistenza agli utenti, 1 referente in ambito applicativo e 1 referente in ambito sistemistico che possano prontamente collaborare con il Fornitore di Attestati Elettronici di Attributi (IPZS per gli Attestati Elettronici di interesse pubblico) e con il Fornitore di Wallet (PagoPA per la soluzione IT-Wallet pubblica) per la risoluzione congiunta di segnalazioni degli utenti e/o malfunzionamenti tra i servizi. 
- Canali: fornisci almeno un canale di assistenza (es. e-mail assistenza, numero di telefono, etc.) di responsabilità dell’Ente che rappresenti, nel ruolo di Titolare di Fonte Autentica, a cui si possa indirizzare l’utente per quelle richieste di supporto e segnalazioni non gestibili all’interno della soluzione IT-Wallet.  

> - FAQ: per ciascuna voce compila **`risposta`** (stringa, booleano o null); **`domanda`** e **`esempio`** sono testi guida preimpostati nello stesso ordine usato per i casi d'uso (`domanda`, poi `risposta` da valorizzare, poi `esempio` con esempio di risposta).

Reviewer: ```suggestion
- FAQ: per ciascuna voce compila `risposta` (stringa, booleano o null); `domanda` e `esempio` sono testi guida preimpostati nello stesso ordine usato per i casi d'uso (`domanda`, poi `esempio` con esempio di risposta, poi `risposta` da valorizzare).
```

- testi informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 

> 1) Poniti le seguenti domande (puoi fare riferimento alla sezione `casi d'uso`, già compilata):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 

Reviewer: ```suggestion
1) Poniti le seguenti domande (puoi fare riferimento alla sezione "Casi d'uso", già compilata):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
```

- **progettare un'esperienza d'uso che apporti valore reale** sia a cittadini che ai verificatori; è utile definire a monte quali potranno essere le occasioni d'uso dell'EAA prodotto con i propri dati, a partire dall'analisi dell'esperienza attuale di fruizione del corrispettivo documento fisico, se esistente (es. si pensi alla modalità di presentazione del codice a barre per l'uso della Tessera Sanitaria o del QR code per la verifica della Carta Europea della Disabilità); 
- **orientare il tipo di formato** con cui il Fornitore di Attestati Elettronici emetterà l'EAA (SD-JWT-VC per scenari in remoto e mdoc-CBOR per scenari in prossimità).

> A tal fine, l’Ente deve compilare la sezione `casi d'uso` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Per riferimenti e istruzioni di compilazione vedi [Appendice A](#appendice-a--casi-duso).

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione `casi d'uso` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vedi [Appendice A](#appendice-a--casi-duso).
```

> A tal fine, l’Ente deve compilare la sezione `casi d'uso` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vedi [Appendice A](#appendice-a--casi-duso).

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione "Casi d'uso" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vedi [Appendice A](#appendice-a--casi-duso).
```

- Validare il modulo compilato utilizzando la funzionalità “valida” in testa al modulo ed esportarlo in formato JSON.

A compilazione, validazione ed export conclusi, l’Ente deve procedere con la sottomissione del modulo secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

L’Ente dovrà creare, rinominare, compilare e validare un modulo per ciascun dataset relativo a un EAA di interesse.

Il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) contiene le seguenti sezioni:

- Denominazioni ufficiali
- Casi d’uso
- Dataset
- Mappatura errori
- Mappatura stati
- Assistenza

> 

Reviewer: ```suggestion
Di seguito le indicazioni di compilazione di ciascuna sezione del modulo.
```

Aggiunta frase (spostata qui da sotto) per risolvere un commento vecchio ma ancora attuale

> Di seguito le indicazioni di compilazione di ciascuna sezione del modulo.

Reviewer: ```suggestion

Di seguito le indicazioni di compilazione di ciascuna sezione del modulo.
```

Aggiunto spazio per andare a capo

> Di seguito le indicazioni di compilazione di ciascuna sezione del modulo.

Reviewer: ```suggestion

```

Elimino perchè spostato sopra

### Denominazioni ufficiali

L’obiettivo della sezione “Denominazioni ufficiali” è quello di definire la denominazione ufficiale con cui si intende veicolare il nome dell’Ente titolare e definire la denominazione ufficiale dell’EAA relativa allo specifico dataset, secondo quanto definito nello Step 1.

#### Destinatari

La sezione “Denominazioni ufficiali” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto.

#### Istruzioni di compilazione

- Compila i campi `nome ente titolare` e `nome eaa` seguendo i suggerimenti di compilazione.

> - Il campo `versione` è il numero di versione del modulo (valore fisso, es. 1.1.0) e non va confuso con una data di compilazione.

Reviewer: ```suggestion
- Il campo `versione` è il numero di versione del modulo (valore fisso, es. 1.1.0) e non va confuso con una data di compilazione.

---
```

aggiunto per avere la riga anche a chiusura di questa sezione

---

### Dataset

L’obiettivo della sezione “Dataset” è quello di supportare gli Enti nella definizione dei parametri necessari al Fornitore di Attestati Elettronici di Attributi per l’identificazione di uno specifico dataset e nella definizione dei dati di risposta ovvero degli specifici campi del dataset restituito dall'e-service (Attestato Elettronico di Attributi).

#### Destinatari

La sezione “Dataset” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT.

#### Istruzioni di compilazione

1. Definisci il valore `dataset_id`, necessario per identificare in maniera univoca uno specifico dataset, utilizzando preferibilmente un identificativo di tipo UUID oppure, in alternativa, adottando una stringa breve, descrittiva e priva di spazi in formato kebab case per nomi composti (es. tesserino-medici);
2. Definisci i `parametri di richiesta`, necessari per l’emissione dell’EAA relativo al dataset attraverso la fruizione dell’e-service;
3. Definisci i `dati di risposta`. In particolare:

>    - Prima di iniziare la compilazione, consulta i Template PDND Data Model e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche;

Reviewer: memo: link "Template PDND Data Model" link be retrieved and added

Author: fatto

>    - Prima di iniziare la compilazione, consulta i [Template PDND Data Model](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/e-service-pdnd-template.html) e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche;

Reviewer: il link, se possibile, dovrebbe essere un link esterno fornito da IPZS che permetta di visualizzare i template resi disponibili

Author: spostato qui: https://github.com/italia/eid-wallet-it-docs/issues/1131

- Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato).

---

### Mappatura stati

L’obiettivo della sezione "Mappatura stati" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.

#### Destinatari

La sezione “Mappatura stati” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la definizione del ciclo vita e la relativa formulazione di contenuti informativi e messaggi).

> 

Reviewer: ```suggestion
#### Istruzioni di compilazione
```

Author: perché vuoi rimuoverlo? Il contenuto a seguire non si riferisce ai destinatari ma alle modalità di compilazione

Reviewer: Propongo non di rimuoverlo ma di aggiungerlo.
Al momento il titolo #### Istruzioni di compilazione è troppo in basso. Deve essere spostato da riga 512 a qui (507) per poter fare da cappello anche al paragrafo "Valori ammessi.." qui sotto

**Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo i valori letterali `Valido`, `Non Valido`, `Sospeso`, `Scaduto`, `Da aggiornare` (nell'ordine fisso previsto dal manuale). Per compatibilità con file già esportati, lo schema accetta anche la variante `Non valido` (v minuscola) al posto di `Non Valido` per il secondo elemento dell'array. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).

Per ciascuno dei **cinque** stati, il modulo prevede una `descrizione` testuale fissa (vincolata dallo schema): personalizza i messaggi rivolti all'utente tramite `azione utente`, `note` e, se utile, il campo opzionale `messaggio` (testo informativo aggiuntivo in IT-Wallet).

> #### Istruzioni di compilazione

Reviewer: ```suggestion

```

*Figura 4: Esempio di resa grafica della Vista di dettaglio di un’EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di descrizioni*

> A tal fine, l’Ente deve compilare la sezione `data model.dataset` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice B](#appendice-b--data-model).

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione `data model` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice B](#appendice-b--data-model).
```

> In conclusione, un'adeguata definizione del Data Model pone le basi per una corretta implementazione dell'e-service da pubblicare su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)) ma è altresì importante considerare e rispettare i seguenti requisiti tecnici: 

Reviewer: ```suggestion
Un'adeguata definizione del Data Model pone le basi per una corretta implementazione dell'e-service da pubblicare su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)) ma è altresì importante considerare e rispettare i seguenti requisiti tecnici: 
```

*Figura 4: Esempio di resa grafica della Vista di dettaglio di un’EAA nella Soluzione Wallet di PagoPA, caratterizzata da attributi di II livello strutturati come lista di descrizioni*

> A tal fine, l’Ente deve compilare la sezione `data model` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, lunghezza massima consentita, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vedi [Appendice B](#appendice-b--data-model).

Reviewer: ```suggestion
A tal fine, l’Ente deve compilare la sezione "Data model" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) relativa a ciascun dataset specificando il DatasetID e riportando gli attributi adeguatamente definiti e i relativi dettagli (es. tipologia, obbligatorietà, formato, ordinamento, etc.). Per riferimenti e istruzioni di compilazione vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).
```

L'e-service messo a disposizione dall'Ente deve prevedere e gestire specifiche situazioni di errore che possono verificarsi nella fase di recupero dei dati da parte del Fornitore di Attestati di Attributi.

> A tal fine, l'Ente deve compilare la sezione `mappatura errori`del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire, consentendo comunque l'aggiunta di eventuali errori specifici, per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](#appendice-c--mappatura-errori).

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `mappatura errori` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire, consentendo comunque l'aggiunta di eventuali errori specifici, per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](#appendice-c--mappatura-errori).
```

> A tal fine, l'Ente deve compilare la sezione `mappatura errori` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire, consentendo comunque l'aggiunta di eventuali errori specifici, per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vedi [Appendice C](#appendice-c--mappatura-errori).

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione "Mappatura errori" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). La mappatura descrive le risposte che il servizio messo a disposizione dovrà obbligatoriamente gestire, consentendo comunque l'aggiunta di eventuali errori specifici, per garantire una corretta informazione all'utente in caso di errori durante l'ottenimento dell'EAA. Per riferimenti e istruzioni di compilazione vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare).
```

> - **Valido**: EAA emesso, nessun segnale di modifica su uno dei suoi attributi o sul suo stato, entro la data di scadenza. L'utente può utilizzarlo in ogni scenario d'uso;

Reviewer: ```suggestion
- **Valido** (valid): EAA emesso senza alcun segnale o nota di criticità verso l’utente; 
```

> - **Sospeso**: EAA temporaneamente non valido, in uno stato di reversibilità. L'utente deve aspettare che lo stato torni ad essere valido (es. Patente ritirata);

Reviewer: ```suggestion
- **Sospeso** (suspended): EAA temporaneamente non valido, in uno stato di reversibilità. L'utente è invitato ad aspettare che lo stato torni ad essere valido (es. Patente ritirata); 
```

> - **Non Valido** (valore letterale del campo `stato` nel modulo): EAA non più valido, in uno stato di irreversibilità. L'utente può solo eliminarlo e/o richiedere l'emissione di nuovo EAA che sovrascriva il precedente (es. Patente annullata).

Reviewer: ```suggestion
- **Non Valido** (invalid): EAA non più valido, in uno stato di irreversibilità. L'utente è invitato adeliminarlo oppure ad effettuare eventuali procedure propedeutiche ad avviare una nuova emissione che sovrascriva il precedente EAA (es. Patente riemessa);
```

> 

Reviewer: ```suggestion
 - **Da aggiornare** (attribute_update): EAA contenente una o più informazioni obsolete. L’utente è invitato ad aggiornare l’EAA attraverso un nuovo processo di emissione. 
```

> Oltre agli stati sopra elencati, è bene specificare che lo stato di un Attestato Elettronico può essere influenzato anche dalla **scadenza amministrativa** e/o dalla **scadenza tecnica**. Rispettivamente l'EAA può, quindi, assumere anche i seguenti stati: 

Reviewer: ```suggestion

Oltre agli stati sopra elencati, è bene specificare che il ciclo di vita di un Attestato Elettronico sè influenzato anche dalla **scadenza tecnica** (definita dalla Fonte Autentica in sinergia con il Fornitore di Attestati Elettronici con l’obiettivo di innescare periodici aggiornamenti automatici dell’EAA per mitigare potenziali rischi di sicurezza) e dalla **scadenza amministrativa** (se resa disponibile tra gli attributi dell’EAA dal Titolare di Fonte Autentica). 

 Nel caso sia prevista una scadenza amministrativa, l’EAA può assumere anche i seguenti stati: 
```

> - **Scaduto**: EAA con data di scadenza amministrativa superata. La scadenza amministrativa può essere definita dal Titolare di Fonte Autentica e, se ritenuta utile o necessaria, deve essere inclusa come attributo all'interno del data model dell'EAA per garantire messaggi informativi all'interno della soluzione IT-Wallet (es. La tua Patente scade tra 30 giorni);

Reviewer: ```suggestion
- **In scadenza**: EAA in prossimità della data di scadenza amministrativa, se resa disponibile dalla Fonte Autentica. L’utente è invitato a compiere eventuali azioni necessarie per poter riottenere l’EAA aggiornato (es. La tua patente di guida è in scadenza, ricordati di rinnovarla presso gli uffici competenti); 
- **Scaduto**: EAA con data di scadenza amministrativa superata, se resa disponibile dalla Fonte Autentica. L’utente è invitato a compiere eventuali azioni necessarie per riottenere l’EAA aggiornato (es. La tua patente di guida è scaduta. Rinnovala presso gli uffici competenti). 
```

> - **Da aggiornare**: EAA con data di scadenza tecnica superata. La scadenza tecnica è definita del Fornitore di Attestati Elettronici di Attributi ed è impostata generalmente a 1 anno o comunque a un valore inferiore o uguale alla data di scadenza amministrativa. Tale scadenza ha l'obiettivo di richiedere un'azione di riemissione esplicita all'utente e mitigare rischi di sicurezza.

Reviewer: ```suggestion

```

> **Corrispondenza con l'API Fonte Autentica (OpenAPI).** Nel modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), la sezione `mappatura stati` ammette per il campo `stato` esattamente questi valori testuali (da usare così come scritti nel JSON): **`Valido`**, **`Non Valido`**, **`Sospeso`** e **`Scaduto`**. Nei contratti OpenAPI dell'e-service Fonte Autentica (es. `OAS3-PDND-AS.yaml`), il campo `status` di ciascun dataset in `attributeClaims` è invece limitato a **`VALID`**, **`INVALID`** e **`SUSPENDED`**: le scadenze si desumono dai metadati e dai claim (`expiry_date`, `last_updated`, `exp`/`nbf`, ecc.) e, lato specifica, i casi *Issued* ed *Expired* rientrano in `VALID` se non sussistono revoca o sospensione. Indicazione operativa: **Non Valido** corrisponde a `INVALID`; **Sospeso** a `SUSPENDED`; **Valido** a `VALID`; **Scaduto** descrive nello Strumento di progettazione la casistica percepita dall'utente quando l'EAA non è più utilizzabile per scadenza, mentre sul canale OpenAPI il dataset resta caratterizzato da uno dei tre `status` precedenti (di norma `VALID` con scadenza nei metadati, oppure `INVALID` se l'Ente imposta la cessazione — inclusa la scadenza amministrativa senza metadato idoneo — come da Specifiche Tecniche).

Reviewer: ```suggestion
**Corrispondenza con l'API Fonte Autentica (OpenAPI).** Nel modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), la sezione "Mappatura stati" ammette per il campo `stato` esattamente questi valori testuali (da usare così come scritti nel JSON): **`Valido`**, **`Non Valido`**, **`Scaduto`** e **`Da aggiornare`** . Nei contratti OpenAPI dell'e-service Fonte Autentica (es. `OAS3-PDND-AS.yaml`), il campo `status` di ciascun dataset in `attributeClaims` è invece limitato a **`VALID`**, **`INVALID`** e **`SUSPENDED`**: le scadenze si desumono dai metadati e dai claim (`expiry_date`, `last_updated`, `exp`/`nbf`, ecc.) e, lato specifica, i casi *Issued* ed *Expired* rientrano in `VALID` se non sussistono revoca o sospensione. Indicazione operativa: **Non Valido** corrisponde a `INVALID`; **Sospeso** a `SUSPENDED`; **Valido** a `VALID`; **Scaduto** descrive nello Strumento di progettazione la casistica percepita dall'utente quando l'EAA non è più utilizzabile per scadenza, mentre sul canale OpenAPI il dataset resta caratterizzato da uno dei tre `status` precedenti (di norma `VALID` con scadenza nei metadati, oppure `INVALID` se l'Ente imposta la cessazione — inclusa la scadenza amministrativa senza metadato idoneo — come da Specifiche Tecniche).
```

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

> A tal fine, l'Ente deve compilare la sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) per definire messaggi, testi informativi aggiuntivi (campo opzionale `messaggio`) e l'applicabilità dei quattro valori **`Valido`**, **`Non Valido`**, **`Sospeso`** e **`Scaduto`**.

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione "Mappatura stati" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) per definire messaggi, testi informativi aggiuntivi (campo opzionale `messaggio`) e l'applicabilità dei quattro valori **`Valido`**, **`Non Valido`**, **`Sospeso`**, **`Scaduto`** e **`Da aggiornare`**.
```

> Per riferimenti e istruzioni di compilazione vedi [Appendice D](#appendice-d--mappatura-stati).

Reviewer: ```suggestion
Per riferimenti e istruzioni di compilazione vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare). 
```

- **Scaduto**: EAA con data di scadenza amministrativa superata. La scadenza amministrativa può essere definita dal Titolare di Fonte Autentica e, se ritenuta utile o necessaria, deve essere inclusa come attributo all'interno del data model dell'EAA per garantire messaggi informativi all'interno della soluzione IT-Wallet (es. La tua Patente scade tra 30 giorni);
- **Da aggiornare**: EAA con data di scadenza tecnica superata. La scadenza tecnica è definita del Fornitore di Attestati Elettronici di Attributi ed è impostata generalmente a 1 anno o comunque a un valore inferiore o uguale alla data di scadenza amministrativa. Tale scadenza ha l'obiettivo di richiedere un'azione di riemissione esplicita all'utente e mitigare rischi di sicurezza.

> **Corrispondenza con l'API Fonte Autentica (OpenAPI).** Nel file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json), l'array `mappatura stati` ammette per il campo `stato` esattamente questi valori testuali: **Valido**, **Non Valido**, **Sospeso** e **Scaduto**. Nei contratti OpenAPI dell'e-service Fonte Autentica (es. `OAS3-PDND-AS.yaml`), il campo `status` di ciascun dataset in `attributeClaims` è invece limitato a **`VALID`**, **`INVALID`** e **`SUSPENDED`**: le scadenze si desumono dai metadati e dai claim (`expiry_date`, `last_updated`, `exp`/`nbf`, ecc.) e, lato specifica, i casi *Issued* ed *Expired* rientrano in `VALID` se non sussistono revoca o sospensione. Indicazione operativa: **Non Valido** corrisponde a `INVALID`; **Sospeso** a `SUSPENDED`; **Valido** a `VALID`; **Scaduto** descrive nello Strumento di progettazione la casistica percepita dall'utente quando l'EAA non è più utilizzabile per scadenza, mentre sul canale OpenAPI il dataset resta caratterizzato da uno dei tre `status` precedenti (di norma `VALID` con scadenza nei metadati, oppure `INVALID` se l'Ente imposta la cessazione — inclusa la scadenza amministrativa senza metadato idoneo — come da Specifiche Tecniche).

Reviewer: ```suggestion
**Corrispondenza con l'API Fonte Autentica (OpenAPI).** Nel modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), la sezione `mappatura stati` ammette per il campo `stato` esattamente questi valori testuali: **Valido**, **Non Valido**, **Sospeso** e **Scaduto**. Nei contratti OpenAPI dell'e-service Fonte Autentica (es. `OAS3-PDND-AS.yaml`), il campo `status` di ciascun dataset in `attributeClaims` è invece limitato a **`VALID`**, **`INVALID`** e **`SUSPENDED`**: le scadenze si desumono dai metadati e dai claim (`expiry_date`, `last_updated`, `exp`/`nbf`, ecc.) e, lato specifica, i casi *Issued* ed *Expired* rientrano in `VALID` se non sussistono revoca o sospensione. Indicazione operativa: **Non Valido** corrisponde a `INVALID`; **Sospeso** a `SUSPENDED`; **Valido** a `VALID`; **Scaduto** descrive nello Strumento di progettazione la casistica percepita dall'utente quando l'EAA non è più utilizzabile per scadenza, mentre sul canale OpenAPI il dataset resta caratterizzato da uno dei tre `status` precedenti (di norma `VALID` con scadenza nei metadati, oppure `INVALID` se l'Ente imposta la cessazione — inclusa la scadenza amministrativa senza metadato idoneo — come da Specifiche Tecniche).

```

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

> A tal fine, l'Ente deve compilare la sezione `mappatura stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) per definire i messaggi e l'applicabilità dei quattro valori **Valido**, **Non Valido**, **Sospeso** e **Scaduto**.

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) per definire i messaggi e l'applicabilità dei quattro valori **Valido**, **Non Valido**, **Sospeso** e **Scaduto**.
```

Per approfondimenti vai alle Specifiche Tecniche, sezione [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html). 

> A tal fine, l'Ente deve compilare la sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) per definire i messaggi e l'applicabilità dei quattro valori **Valido**, **Non Valido**, **Sospeso** e **Scaduto**.

Author: Scaduto included

### **Definire i contenuti per l'informazione e l'assistenza all'utente**

L'Ente deve contribuire al [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet rendendo disponibili contenuti utili alla predisposizione di nuove Domande Frequenti e/o testi informativi in app, e fornendo i recapiti necessari per la gestione dell'assistenza agli utenti.  

> A tal fine, l'Ente deve compilare la sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vedi [Appendice E](#appendice-e--assistenza).

Reviewer: ```suggestion
A tal fine, l'Ente deve compilare la sezione "Assistenza" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Per riferimenti e istruzioni di compilazione vai all'apposita sezione del capitolo [Modulo da compilare](#modulo-da-compilare). 
```

L'Ente interessato a personalizzare la resa grafica dell'EAA prodotto dai propri dati deve trasmettere i propri materiali come documentazione allegata all'e-service su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)).
### **Validare il modulo “Progettazione caratteristiche EAA”**

A conclusione delle azioni sopra elencate, per proseguire con gli step successivi, l’Ente deve:
-  validare il modulo Progettazione caratteristiche EAA compilato in tutte le sue parti utilizzando la funzionalità “valida” visibile in testa al modulo, che permette di validare il JSON Schema di cui è composto il modulo, e la corretta compilazione di tutti i campi; 
- esportare il modulo compilato e validato in formato JSON. L'export del modulo in formato JSON è funzionale a quanto previsto nello Step 2. 

> Per poter proseguire con gli step successivi, è infatti obbligatorio eseguire validazione JSON Schema e controllo sintattico. Per i comandi e il workflow, vedi [Validazione JSON Schema e Linter](validazione-json-schema-linter.md). La checklist prevede: 

Reviewer: ```suggestion

```

- validazione JSON Schema superata; 
- JSON Linter senza errori; 

> - tutti i campi obbligatori (risposta) compilati per la sezione scelta; 

Reviewer: ```suggestion

```

L'Ente interessato a personalizzare la resa grafica dell'EAA prodotto dai propri dati deve trasmettere i propri materiali come documentazione allegata all'e-service su PDND (vedi [Step 2](#step-2--pubblicazione-in-collaudo)).

> ### **Validare il file “progettazione_EAA”**

Reviewer: ```suggestion
### **Validare il modulo “Progettazione caratteristiche EAA”**
```

> A conclusione delle azioni sopra elencate, l’Ente deve validare il file ”progettazione_EAA" in tutte le sue parti (sezioni `casi d'uso`, `pdnd metadata`, `request`, `data model`, `mappatura errori`, `mappatura stati`, e `assistenza`) utilizzando lo strumento indicato nel file ”validazione_progettazione_caratteristiche_EAA”. 

Reviewer: ```suggestion
A conclusione delle azioni sopra elencate, per proseguire con gli step successivi, l’Ente deve:
-  validare il modulo Progettazione caratteristiche EAA compilato in tutte le sue parti utilizzando la funzionalità “valida” visibile in testa al modulo, che permette di validare il JSON Schema di cui è composto il modulo, e la corretta compilazione di tutti i campi; 
- esportare il modulo compilato e validato in formato JSON. L'export del modulo in formato JSON è funzionale a quanto previsto nello Step 2. 
```

- [sezione e-service del Manuale Operativo PDND Interoperabilità](https://docs.pagopa.it/interoperabilita-1/manuale-operativo/e-service)
- [sezione Titolare di Fonte Autentica delle Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/authentic-sources.html)

> Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare, come documentazione aggiuntiva su PDND, il file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) precedentemente compilato in tutte le sue parti e e validato (sezioni `casi d'uso`, `pdnd metadata`, `request`, `data model`, `mappatura errori`, `mappatura stati`, e `assistenza`). 

Reviewer: ```suggestion
Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare, come documentazione aggiuntiva su PDND, il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) precedentemente compilato in tutte le sue parti, validato ed esportato in formato JSON. 
```

Nel caso di EAA di interesse pubblico, l'Ente deve aggiungere l'attributo certificato Istituto Poligrafico e Zecca dello Stato S.P.A. all'e-service erogato, se possibile impostando l'accettazione automatica delle richieste di fruizione.

Si consiglia di nominare l'e-service in "Creazione EAA [Nome / Nome tipologia EAA] – IT-Wallet" (es. "Creazione EAA Patente di guida – IT-Wallet" oppure "Creazione EAA Titoli di studio – IT-Wallet") e di predisporre una descrizione in linea con le Linee Guida sull’infrastruttura tecnologica della PDND per l’interoperabilità dei sistemi informativi e delle basi di dati ([Allegato 7 - Regole di popolamento](https://italia.github.io/pdnd-guida-nomenclatura-eservice/index.html)) emanate da AgID. 

### **Attivare il servizio Signal Hub in collaudo**

> L'Ente deve attivare in collaudo il servizio [Signal Hub](https://developer.pagopa.it/pdnd-interoperabilita/guides/manuale-operativo-pdnd-interoperabilita/v1.0/riferimenti-tecnici/signal-hub) di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json).  

Reviewer: ```suggestion
L'Ente deve attivare in collaudo il servizio [Signal Hub](https://developer.pagopa.it/pdnd-interoperabilita/guides/manuale-operativo-pdnd-interoperabilita/v1.0/riferimenti-tecnici/signal-hub) di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).
```

Contestualmente al flusso di pubblicazione dell'e-service, l'Ente deve allegare, come documentazione aggiuntiva su PDND, il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) precedentemente compilato in tutte le sue parti, validato ed esportato in formato JSON. 

Nel caso di EAA di interesse pubblico, l'Ente deve aggiungere l'attributo certificato Istituto Poligrafico e Zecca dello Stato S.P.A. all'e-service erogato, se possibile impostando l'accettazione automatica delle richieste di fruizione.

Si consiglia di nominare l'e-service in "Creazione EAA [Nome / Nome tipologia EAA] – IT-Wallet" (es. "Creazione EAA Patente di guida – IT-Wallet" oppure "Creazione EAA Titoli di studio – IT-Wallet") e di predisporre una descrizione in linea con le Linee Guida sull’infrastruttura tecnologica della PDND per l’interoperabilità dei sistemi informativi e delle basi di dati ([Allegato 7 - Regole di popolamento](https://italia.github.io/pdnd-guida-nomenclatura-eservice/index.html)) emanate da AgID. 

### **Attivare il servizio Signal Hub in collaudo**

> L'Ente deve attivare in collaudo il servizio [Signal Hub](https://developer.pagopa.it/pdnd-interoperabilita/guides/manuale-operativo-pdnd-interoperabilita/v1.0/riferimenti-tecnici/signal-hub) di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).

Reviewer: ```suggestion
L'Ente deve attivare in collaudo il servizio [Signal Hub](https://developer.pagopa.it/pdnd-interoperabilita/guides/manuale-operativo-pdnd-interoperabilita/v1.0/riferimenti-tecnici/signal-hub) di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione "Mappatura stati" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).
```

### **Attivare il servizio Signal Hub in produzione**

> L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura stati` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json).  

Reviewer: ```suggestion
L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).
```

### **Attivare il servizio Signal Hub in produzione**

> L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).

Reviewer: ```suggestion
L'Ente deve attivare in produzione il servizio Signal Hub di PDND per il relativo e-service, in coerenza con quanto definito dalle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/signal-hub-endpoint.html) per la gestione degli stati, precedentemente mappati nella sezione "Mappatura stati" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa).
```

L'Ente deve garantire il corretto funzionamento dell'e-service nel tempo, programmare adeguate azioni di monitoraggio e aggiornamento se richieste, ad esempio, da cambiamenti normativi o procedurali (es. nuovi dati, stati, casistiche di errore, etc.).

È altresì importante che l’ente mantenga il proprio e-service aggiornato all’ultima versione indicata dalle Specifiche Tecniche IT-Wallet, facendo riferimento al branch LTS e assicurandosi in particolare di implementare sempre l’ultima versione di patch disponibile.

### **Gestire problematiche e fornire assistenza agli utenti**

> L'Ente deve garantire un costante aggiornamento delle informazioni riportate delle informazioni riportate nella sezione `assistenza` del file [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json) al fine di:

Reviewer: ```suggestion
L'Ente deve garantire un costante aggiornamento delle informazioni riportate delle informazioni riportate nella sezione "Assistenza" del modulo [Progettazione caratteristiche EAA(https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) al fine di:
```

- **Contribuire alla risoluzione di bug** 

> Il referente dell'ambito sistemistico e il referente dell'ambito applicativo, così come definito nella sezione `assistenza` del modulo, devono contribuire alla diagnosi congiunta delle segnalazioni ricevute da Fornitore di Attestati Elettronici di Attributi (IPZS, nel caso di EAA di interesse pubblico) e Fornitori di Wallet (PagoPA, nel caso della soluzione pubblica IT-Wallet) e relativa risoluzione, secondo quanto definito dal [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet. 

Reviewer: ```suggestion
Il referente dell'ambito sistemistico e il referente dell'ambito applicativo, così come definito nella sezione "Assistenza" del modulo, devono contribuire alla diagnosi congiunta delle segnalazioni ricevute da Fornitore di Attestati Elettronici di Attributi (IPZS, nel caso di EAA di interesse pubblico) e Fornitori di Wallet (PagoPA, nel caso della soluzione pubblica IT-Wallet) e relativa risoluzione, secondo quanto definito dal [modello di assistenza](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#assistenza-utente) del Sistema IT-Wallet. 
```

- **Garantire il supporto agli utenti** 

> Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.

Reviewer: ```suggestion
Il referente per l'ambito assistenza ed almeno un canale di contatto dedicato agli utenti finali (es. indirizzo e-mail, numero telefonico, etc.), così come definito nella sezione "Assistenza" del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa), devono sempre essere disponibili per gestire eventuali problemi relativi all'EAA, come ad esempio la segnalazione di dati errati o di errori nella fase di ottenimento dell'EAA da parte dell'utente.
```

## Modulo da compilare

Per assolvere a quanto previsto dallo Step 1, l‘Ente deve: 
- Accedere al modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa);

> - rinominare il modulo in "progettazione_ caratteristiche_eaa_[Nome Ente Titolare]_[Nome EAA]" (es. "progettazione_eaa_mim_titolo_di_studio");

Reviewer: ```suggestion
- Rinominare il modulo in "Caratteristiche EAA - [Nome EAA] - [Nome Ente Titolare]" (es. "Caratteristiche EAA – Titolo di studio - MIM");
```

> - compilare il modulo in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di casi d'uso, dati, mappatura errori, stati e assistenza. E' possibile salvare localmente nel web browser un modulo in fase di compulazione ed è possibile l'import di JSON/CSV;

Reviewer: ```suggestion
- Compilare il modulo in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di casi d'uso, dati, mappatura errori, stati e assistenza. E' possibile salvare localmente nel web browser un modulo in fase di compulazione ed è possibile l'import di JSON/CSV. Nel caso in cui più soggetti siano incaricati della compilazione del modulo, è quindi possibile esportare e condividere una bozza parzialmente compilata affinché possa essere successivamente caricata e completata da un altro soggetto. È inoltre possibile lavorare in parallelo su copie distinte del modulo; in tal caso, è necessario assicurarsi che tutti i contributi confluiscano in un’unica versione finale completa in ogni sua parte; 
```

> - validare il modulo compilato utilizzando la funzionalità “valida” in testa al modulo ed esportarlo in formato JSON.

Reviewer: ```suggestion
- Validare il modulo compilato utilizzando la funzionalità “valida” in testa al modulo ed esportarlo in formato JSON.
```

> L’ente dovrà creare, rinominare, compilare, validare ed esportare un modulo per ciascuna EAA di interesse. 

Reviewer: ```suggestion

```

> A compilazione, validazione ed export conclusi, l’Ente può procedere con la sottomissione del modulo secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

Reviewer: ```suggestion
A compilazione, validazione ed export conclusi, l’Ente deve procedere con la sottomissione del modulo secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

L’ente dovrà creare, rinominare, compilare e validare un modulo per ciascun dataset relativo a un EAA di interesse.
```

Il modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa) contiene le seguenti sezioni: 

- Casi d’uso (istruzioni di compilazione in [Appendice A](#appendice-a--casi-duso)) 
- Data Model (istruzioni di compilazione in [Appendice B](#appendice-b--data-model)) 
- Lista attributi (opzionale; vedi [Appendice B](#appendice-b--data-model))
- Mappatura errori (istruzioni di compilazione in [Appendice C](#appendice-c--mappatura-errori)) 
- Mappatura stati (istruzioni di compilazione in [Appendice D](#appendice-d--mappatura-stati)) 

> - Assistenza (istruzioni di compilazione in [Appendice E](#appendice-e--assistenza)) 

Reviewer: ```suggestion
- Denominazioni ufficiali
- Casi d’uso
- Data Model
- Mappatura errori
- Mappatura stati 
- Assistenza
```

> 

Reviewer: ```suggestion
### Denominazioni ufficiali 
 
L’obiettivo della sezione “Denominazioni ufficiali” è quello di definire la denominazione ufficiale con cui si intende veicolare il nome dell’Ente titolare e definire la denominazione ufficiale dell’EAA relativa allo specifico dataset, secondo quanto definito nello Step 1. 

Destinatari: 
La sezione “Denominazioni ufficiali” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto. 
 
Istruzioni di compilazione: 
- Compila i campi `nome ente titolare `e `nome eaa` seguendo i suggerimenti di compilazione.  
- Il campo `versione` è il numero di versione del modulo (valore fisso, es. 1.1.0) e non va confuso con una data di compilazione. 
```

Reviewer: Commento superato. (ho provato a risolverlo ma mi da "failed to resolve")

> ## Appendice A – Casi d'uso EAA

Reviewer: ```suggestion
### Casi d'uso 
```

> Questa appendice descrive le istruzioni di compilazione della sezione `casi d'uso` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire.

Reviewer: ```suggestion

```

> ### Obiettivo

Reviewer: ```suggestion

```

> L’obiettivo della sezione `casi d'uso` è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti. 

Reviewer: ```suggestion
L’obiettivo della sezione "Casi d'uso" è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti. 
```

> 

Reviewer: ```suggestion
Destinatari: 
La sezione “Casi d’uso” si rivolge ad utenti funzionari, amministrativi, dirigenti o responsabili di prodotto. 
```

Reviewer: Commento superato. (ho provato a risolverlo ma mi da "failed to resolve")

> ### Istruzioni di compilazione 

Reviewer: ```suggestion
Istruzioni di compilazione:
```

> ### Istruzioni di compilazione 

Reviewer: ```suggestion
Istruzioni di compilazione:
```

> 1. **Denominazioni ufficiali per Ente e EAA**: nella sezione `metadata` compila `nome ente titolare` e `nome eaa`; il campo `versione` è il numero di versione del template di progettazione (valore fisso indicato nello schema, es. `1.1.0`) e non va confuso con una data di compilazione. 

Reviewer: ```suggestion

```

> 2. **domande pertinenti**: nel caso l'EAA non si riferisca a un documento fisico o digitale già esistente (es. patente), non compilare le domande che iniziano con "In caso di documento già esistente..";

Reviewer: ```suggestion
1. **Domande pertinenti**: nel caso l'EAA non si riferisca a un documento fisico o digitale già esistente (es. patente), non compilare le domande che iniziano con "In caso di documento già esistente..";
```

> 3. **Domande a risposta aperta o sì/no**: ogni voce ha sempre tre campi; nell'interfaccia del modulo l'ordine è **`domanda`** (testo fisso), **`risposta`** (da compilare: stringa, oppure `true`/`false`/`null` in JSON per sì/no o bozza) e **`esempio`** (testo guida con esempio di risposta). Il campo `esempio` non sostituisce la `risposta`.

Reviewer: ```suggestion
2. **Domande a risposta aperta o chiusa**: ogni voce ha tre campi; nell'interfaccia del modulo l'ordine è **`domanda`** (testo fisso), **`esempio`** (testo guida con esempio di risposta) e **`risposta`** (risposta aperta o risposta chiusa). Il campo `esempio` non sostituisce la `risposta`.
```

> 4. **Nomi delle proprietà nel JSON**: usa le chiavi esattamente come nello schema di validazione (parole separate da **un solo spazio**; evita doppi spazi, ad esempio `canali richiesta doc preesistente`, `richieste fisico vs digitale doc preesistente`, `normativa doc preesistente`).

Reviewer: ```suggestion

```

> **Esempio di compilazione** (fragmento):

Reviewer: ```suggestion

```

> ```json

Reviewer: ```suggestion

```

{

>   "metadata": {

Reviewer: ```suggestion

```

    "nome ente titolare": "Ministero dell'Istruzione",
    "nome eaa": "Titolo di studio",
    "versione": "1.1.0"
  },
  "casi d'uso": {
    "target utenti": {
      "utenti doc preesistente": {
        "domanda": "Chi può ad oggi richiedere il documento? ...",
        "risposta": "Tutti i cittadini italiani in possesso di diploma rilasciato da istituto italiano",
        "esempio": "Solo maggiorenni residenti in una specifica regione / ..."
      }
    }
  }

> }

Reviewer: ```suggestion

```

> ```

Reviewer: ```suggestion

```

---

> ## Appendice B – Data Model

Reviewer: ```suggestion
### Data Model
```

> Questa appendice descrive le istruzioni di compilazione della sezione `data model` del modulo (https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

Reviewer: ```suggestion

```

> **Obiettivo**

Reviewer: ```suggestion

```

> L’obiettivo della sezione `data model` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di dati resi disponibili al suo interno. 

Reviewer: ```suggestion
L’obiettivo della sezione "Data model" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di dati resi disponibili al suo interno. 
```

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Data model” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la definizione del tipo di dati e dell’ordinamento). 
```

Reviewer: commento superato

> **Istruzioni di compilazione**

Reviewer: ```suggestion
Istruzioni di compilazione:
```

> 

Reviewer: ```suggestion
1. Prima di iniziare la compilazione, consulta i Template PDND Data Model e usali come punto di partenza per il tuo e-service così da assicurare un'elevata aderenza e compliance alle Specifiche Tecniche;
```

"Template PDND Data Model" link to be added

Reviewer: Modifica superata

> 1. Associa a ciascun dato che si intende rende disponibile all’interno dell’EAA un "nome campo", assicurandoti che sia parlante e che descriva adeguatamente il dato. 

Reviewer: ```suggestion
2. Definisci il valore dataset_id, necessario per identificare in maniera univoca uno specifico dataset, utilizzando un identificativo di tipo UUID oppure, in alternativa, adottando una stringa breve, descrittiva e priva di spazi in formato kebab case per nomi composti (es. tesserino-medici); 
3. Associa a ciascun dato che si intende rende disponibile all’interno dell’EAA un "nome campo", visibile all’utente finale, assicurandoti che sia parlante e che descriva adeguatamente il dato; 
```

> 2. Ordina i campi in modo da facilitare la leggibilità: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato.

Reviewer: ```suggestion
4. Ordina i campi in modo da facilitare la leggibilità da parte dell’utente finale: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato. 
```

> 3. (Opzionale) Compila la sezione di primo livello **`lista attributi`** del modulo JSON: è una tabella di supporto (categoria, parametro, nome campo, descrizione) per organizzare e documentare come presentare gli attributi in app (coerente con le viste a «lista di attributi» descritte in questa appendice); se non ti è utile, puoi omettere la proprietà o lasciare le righe vuote dove consentito dallo schema.

Reviewer: ```suggestion

```

---

> ## Appendice C – Mappatura errori

Reviewer: ```suggestion
### Mappatura errori
```

> Questa appendice descrive le istruzioni di compilazione della sezione `mappatura errori` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

Reviewer: ```suggestion

```

> **Obiettivo**

Reviewer: ```suggestion

```

> L’obiettivo della sezione `mappatura errori` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.

Reviewer: ```suggestion
L’obiettivo della sezione "Mappatura errori" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di errori che potrebbero occorrere interagendo con l'e-service corrispondente.
```

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Mappatura errori” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la formulazione di contenuti informativi e messaggi). 
```

Reviewer: commento superato

> **Istruzioni di compilazione**

Reviewer: ```suggestion
Istruzioni di compilazione:
```

> - scaricare il file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json)

Reviewer: ```suggestion
- Accedere al modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa);
```

> - duplicare il file JSON per ciascuna EAA di interesse e rinomina ciascun file in "progettazione_ caratteristiche_eaa_[Nome Ente Titolare]_[Nome EAA]" (es. "progettazione_eaa_mim_titolo_di_studio.json"). 

Reviewer: ```suggestion
- rinominare il modulo in "progettazione_ caratteristiche_eaa_[Nome Ente Titolare]_[Nome EAA]" (es. "progettazione_eaa_mim_titolo_di_studio");
```

> - compilare ciascun file JSON in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di dati esposti, casi d'uso, assistenza, mappatura errori e stati.  

Reviewer: ```suggestion
- compilare il modulo in tutte le sue parti al fine di definire a monte tutte le caratteristiche che assumerà l’Attestato Elettronico emesso dal Fornitore di Attestati Elettronici di Attributi a livello di casi d'uso, dati, mappatura errori, stati e assistenza. E' possibile salvare localmente nel web browser un modulo in fase di compulazione ed è possibile l'import di JSON/CSV;
```

> - compilare e validare ciascun file JSON usando l'utility web [IT-Wallet Webforms](https://italia.github.io/eid-wallet-it-forms/), pubblicata open source su [GitHub](https://github.com/italia/eid-wallet-it-forms): è sufficiente compilare i campi del form, con possibilità di salvataggio locale nel web browser e di import/export JSON/CSV, includendo la validazione del contenuto rispetto allo schema.

Reviewer: ```suggestion
- validare il modulo compilato utilizzando la funzionalità “valida” in testa al modulo ed esportarlo in formato JSON.
```

> 

Reviewer: ```suggestion
L’ente dovrà creare, rinominare, compilare, validare ed esportare un modulo per ciascuna EAA di interesse. 
```

> A compilazione e validazione conclusa, l’Ente può procedere con la sottomissione del file secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni di questa sezione secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).

Reviewer: ```suggestion
A compilazione, validazione ed export conclusi, l’Ente può procedere con la sottomissione del modulo secondo quanto descritto nello [Step 2](#step-2--pubblicazione-in-collaudo), ricordando di mantenere sempre aggiornate le informazioni secondo le modalità definite nello [Step 7](#step-7--manutenzione-e-assistenza).
```

- Casi d’uso (istruzioni di compilazione in [Appendice A](#appendix-a)) 
- Data Model (istruzioni di compilazione in [Appendice B](#appendix-b)) 
- Mappatura errori (istruzioni di compilazione in [Appendice C](#appendix-c)) 
- Mappatura stati (istruzioni di compilazione in [Appendice D](#appendice-d--mappatura-stati)) 
- Assistenza (istruzioni di compilazione in [Appendice E](#appendix-e)) 

## Appendice A – Casi d'uso EAA

> Questa appendice descrive le istruzioni di compilazione della sezione `casi d'uso` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire.

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `casi d'uso` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire.
```

### Obiettivo

L’obiettivo della sezione `casi d'uso` è quello di supportare gli Enti nella definizione dei casi d'uso e delle modalità di utilizzo degli EAA di cui intendono fornire i dati, a partire dall'analisi delle attuali modalità di utilizzo dei corrispettivi documenti, ove esistenti. 

### Istruzioni di compilazione 

> 1. **metadata**: inserisci `nome ente titolare`, `nome eaa` e ;

Reviewer: ```suggestion
1. **Denominazioni ufficiali per Ente e EAA**: inserisci `nome ente titolare`, `nome eaa` e data_compilazione (formato ISO: AAAA-MM-GG); 
```

> 2. **campi opzionali** e **campi obbligatori**: nel caso l'EAA si riferisce a un documento fisico o digitale già esistente (es. patente) compila sia i campi obbligatori che i campi opzionali (`volume_rilascio`, `canali_richiesta`, `formato_attuale,` `pagamento`, ecc.). Nel caso invece l'EAA non abbia un documento preesistente corrispondente, compila solo i campi obbligatori e lascia vuoti i campi opzionali;

Reviewer: ```suggestion
2. **domande pertinenti**: nel caso l'EAA non si riferisca a un documento fisico o digitale già esistente (es. patente), non compilare le domande che iniziano con "In caso di documento già esistente..";
```

> 3. **risposta**: compila il campo `risposta` rispondendo in maniera chiara ed esaustiva ad ogni domanda; il campo `suggerimento` è solo indicativo.

Reviewer: ```suggestion
3. **risposta**: compila il campo `risposta` rispondendo in maniera chiara ed esaustiva ad ogni domanda; il campo `esempio` è solo indicativo.
```

---
## Appendice B – Data Model

> Questa appendice descrive le istruzioni di compilazione dell’array `dataset` in `data model` (data model di risposta dell’e-service) nel file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `data model` del modulo (https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 
```

**Obiettivo**

> L’obiettivo della sezione `data model.dataset` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di dati resi disponibili al suo interno. 

Reviewer: ```suggestion
L’obiettivo della sezione `data model` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di dati resi disponibili al suo interno. 
```

> 3. Ordina i campi in modo da facilitare la leggibilità: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato.

Reviewer: ```suggestion
2. Ordina i campi in modo da facilitare la leggibilità: inserisci per primi i dati anagrafici (nome, cognome, data di nascita, luogo di nascita, codice fiscale), poi i dati specifici dell'attestato.
```

> ```json

Reviewer: ```suggestion

```

}

> ```

Reviewer: ```suggestion

```

> *Tabella 1 – Struttura Data Model (sezione `data model.dataset`)*

Reviewer: ```suggestion

```

> ```json

Reviewer: ```suggestion

```

      "codice": 541,
      "esito": "EAA in stato non valido o sospeso",

>       "applicabile": false,

Reviewer: ```suggestion

```

> *Tabella 3 – Mappatura Errori (sezione `mappatura errori`)*

Reviewer: ```suggestion

```

> ## Appendice D – Mappatura stati

Reviewer: ```suggestion
### Mappatura stati
```

> Questa appendice descrive le istruzioni di compilazione della sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

Reviewer: ```suggestion

```

> **Obiettivo**

Reviewer: ```suggestion

```

> L’obiettivo della sezione `mappatura stati` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.

Reviewer: ```suggestion
L’obiettivo della sezione "Mappatura stati" è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.
```

> **Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo i valori letterali **`Valido`**, **`Non Valido`**, **`Sospeso`**, **`Scaduto`** (nell'ordine fisso previsto dal template). Per compatibilità con file già esportati, lo schema accetta anche la variante **`Non valido`** (v minuscola) al posto di **Non Valido** per il secondo elemento dell'array. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).

Reviewer: ```suggestion
**Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo i valori letterali **`Valido`**, **`Non Valido`**, **`Sospeso`**, **`Scaduto`**, **`Da aggiornare`** (nell'ordine fisso previsto dal template). Per compatibilità con file già esportati, lo schema accetta anche la variante **`Non valido`** (v minuscola) al posto di **Non Valido** per il secondo elemento dell'array. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).
```

> Per ciascuno dei quattro stati, il modulo prevede una **descrizione** testuale fissa (vincolata dallo schema): personalizza i messaggi rivolti all'utente tramite **`azione utente`**, **`note`** e, se utile, il campo opzionale **`messaggio`** (testo informativo aggiuntivo in IT-Wallet).

Reviewer: ```suggestion

```

> 

Reviewer: ```suggestion
Destinatari: 

La sezione “Mappatura stati” si rivolge ad utenti con profilo tecnico, tra cui sviluppatori, referenti dei Sistemi Informativi, partner tecnologici o fornitori IT. Per la definizione di alcuni aspetti, si suggerisce il coinvolgimento di utenti con profilo amministrativo o dirigenziale (es. per la definizione del ciclo vita e la relativa formulazione di contenuti informativi e messaggi). 
```

> **Istruzioni di compilazione**

Reviewer: ```suggestion
Istruzioni di compilazione:
```

> 1. Mappa la condizione di applicabilità di ciascuno stato relativamente all'attestato in analisi (`applicabile`: `true` / `false`);

Reviewer: ```suggestion
1. Mappa la condizione di applicabilità di ciascuno stato relativamente all'attestato in analisi (applicabile: `true` / `false`);
```

Reviewer: commento minor vecchio, superabile

> 2. Il campo **descrizione** per ciascuno stato è quello predefinito del template (vincolato dallo schema e non modificabile); usa **`azione utente`**, **`note`** e opzionalmente **`messaggio`** per chiarire al cittadino cosa fare;

Reviewer: ```suggestion
2. Il campo descrizione per ciascuno stato è quello predefinito del modulo; 
```

> 3. Definisci l'**azione utente** da condividere con l'utente (es. «I tuoi dati sono stati aggiornati nella banca dati ANIS, scarica la nuova versione digitale del documento»). Usa il campo **note** per ulteriori indicazioni operative;

Reviewer: ```suggestion
3. Definisci la **`causa`** del cambiamento di stato e l'**`azione utente`** ovvero il messaggio da comunicare all’utente in caso di cambiamento stato (es. “I tuoi dati sono stati aggiornati nella banca dati ANIS, scarica la nuova versione digitale del documento”). Usa opzionalmente il campo **`note`** per aggiungere ulteriori informazioni utili o indicazioni operative. 
```

> 4. Se serve un messaggio dedicato in app oltre a descrizione e azione utente, valorizza opzionalmente **messaggio**; altrimenti ometti il campo o lascialo vuoto;

Reviewer: ```suggestion

```

> 5. Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).

Reviewer: ```suggestion
Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).
```

Estratto strutturale (nomi campo come nel JSON del modulo; `messaggio` è opzionale):

> ```json

Reviewer: ```suggestion

```

"mappatura stati": [
  {
    "stato": "Valido",
    "descrizione": "L'EAA è valido e può essere utilizzato",
    "applicabile": true,
    "azione utente": "",
    "note": ""
  },
  {
    "stato": "Non Valido",
    "descrizione": "L'EAA non è più valido e dunque non può essere più utilizzato",
    "applicabile": false,
    "azione utente": "L'utente deve scaricare nuovamente l'EAA in app",
    "note": ""
  },
  {
    "stato": "Sospeso",
    "descrizione": "L'EAA è sospeso e non può essere temporaneamente utilizzato",
    "applicabile": false,
    "azione utente": "",
    "note": ""
  },
  {
    "stato": "Scaduto",
    "descrizione": "L'EAA ha superato la data di scadenza amministrativa e richiede rinnovo o nuova emissione",
    "applicabile": false,
    "azione utente": "",
    "note": ""
  }

> ]

Reviewer: ```suggestion

```

```

> ## Appendice E – Assistenza

Reviewer: ```suggestion
### Assistenza
```

> Questa appendice descrive le istruzioni di compilazione della sezione `assistenza` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 

Reviewer: ```suggestion

```

> **Obiettivo** 

Reviewer: ```suggestion

```

}
```

> *Tabella 2 – Lista nome campo (sezione `lista nome campo`)*

Reviewer: ```suggestion

```

## Appendice C – Mappatura errori

> Questa appendice descrive le istruzioni di compilazione della sezione `mappatura errori` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `mappatura errori` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 
```

> - Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato, annullato). 

Reviewer: ```suggestion
- Per l'errore 541 (EAA in stato non valido o sospeso), descrivi la causa secondo le [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/en/OAS3-PDND-Issuer.html#tag/e-services-PDND/operation/notifyStatusCredentials) (es. scaduto, sospeso, revocato). 
```

> ```json

Reviewer: ```suggestion

```

- Referenti: fornisci e mantieni aggiornati, i dati di contatto (nome, cognome, email, telefono) di almeno 1 referente per l’assistenza agli utenti, 1 referente in ambito applicativo e 1 referente in ambito sistemistico che possano prontamente collaborare con il Fornitore di Attestati Elettronici di Attributi (IPZS per gli Attestati Elettronici di interesse pubblico) e con il Fornitore di Wallet (PagoPA per la soluzione IT-Wallet pubblica) per la risoluzione congiunta di segnalazioni degli utenti e/o malfunzionamenti tra i servizi. 
- Canali: fornisci almeno un canale di assistenza (es. e-mail assistenza, numero di telefono, etc.) di responsabilità dell’Ente che rappresenti, nel ruolo di Titolare di Fonte Autentica, a cui si possa indirizzare l’utente per quelle richieste di supporto e segnalazioni non gestibili all’interno della soluzione IT-Wallet.  

> - FAQ: per ciascuna voce compila **`risposta`** (stringa, booleano o null); **`domanda`** e **`esempio`** sono testi guida preimpostati nello stesso ordine usato per i casi d'uso (`domanda`, poi `risposta` da valorizzare, poi `esempio` con esempio di risposta).

Reviewer: ```suggestion
- FAQ: per ciascuna voce compila **`risposta`** (stringa, booleano o null); **`domanda`** e **`esempio`** sono testi guida preimpostati nello stesso ordine usato per i casi d'uso (`domanda`, poi `esempio` con esempio di risposta, poi `risposta` da valorizzare).
```

> - testi informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 

Reviewer: ```suggestion
- Testi informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 
```

> 1) Poniti le seguenti domande (puoi fare riferimento alla sezione `casi d'uso`, già compilata):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 

Reviewer: ```suggestion
1) Poniti le seguenti domande (puoi fare riferimento alla sezione "Casi d'uso"  già compilata):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
```

}

> ```

Reviewer: ```suggestion

```

*Tabella 3 – Mappatura Errori (sezione `mappatura errori`)*

## Appendice D – Mappatura stati

> Questa appendice descrive le istruzioni di compilazione della sezione `mappatura stati` del file JSON [Progettazione caratteristiche EAA](progettazione-caratteristiche-eaa.json). Assicurati di aver letto quanto riportato nella sezione [File da compilare](#file-da-compilare) prima di proseguire. 

Reviewer: ```suggestion
Questa appendice descrive le istruzioni di compilazione della sezione `mappatura stati` del modulo [Progettazione caratteristiche EAA](https://italia.github.io/eid-wallet-it-forms/form.html?webform=authentic-sources-eaa). Assicurati di aver letto quanto riportato nella sezione [Modulo da compilare](#modulo-da-compilare) prima di proseguire. 
```

**Obiettivo**

L’obiettivo della sezione `mappatura stati` è quello di supportare gli Enti nella definizione delle caratteristiche dell'e-service - e quindi del corrispettivo Attestato Elettronico di Attributi (EAA) - in termini di stati che potrebbero caratterizzare l’EAA nel corso del suo ciclo di vita.

> **Valori ammessi per `stato` e uso in OpenAPI.** Nel JSON di progettazione il campo `stato` di ogni elemento dell'array può assumere solo: **Valido**, **Non Valido**, **Sospeso**, **Scaduto** (come da schema di validazione). Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).

Reviewer: ```suggestion
**Valori ammessi per `stato` e uso in OpenAPI.** Il campo `stato` di ogni elemento dell'array può assumere solo: **Valido**, **Non Valido**, **Sospeso**, **Scaduto**. Sul canale tecnico, il campo `status` dei dataset nell'OpenAPI Fonte Autentica (`OAS3-PDND-AS.yaml`) resta limitato a **VALID**, **INVALID**, **SUSPENDED**; per la correlazione con i metadati di scadenza vedi il paragrafo *Corrispondenza con l'API Fonte Autentica* in [Step 1 | Progettazione caratteristiche EAA](#step-1--progettazione-caratteristiche-eaa).
```

4. Per approfondimenti: [Ciclo di Vita degli Attestati Elettronici](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/credential-revocation.html).

> ```json

Reviewer: ```suggestion

```

}

> ```

Reviewer: ```suggestion

```

> *Tabella 4 – Mappatura stati (sezione `mappatura stati`)*

Reviewer: ```suggestion

```

- Canali: fornisci almeno un canale di assistenza (es. e-mail assistenza, numero di telefono, etc.) di responsabilità dell’Ente che rappresenti, nel ruolo di Titolare di Fonte Autentica, a cui si possa indirizzare l’utente per quelle richieste di supporto e segnalazioni non gestibili all’interno della soluzione IT-Wallet.  
- FAQ: contribuisci alla definizione dei contenuti utili a predisporre le domande più frequenti e comuni relative all’EAA in analisi, dall’emissione all’utilizzo fino alla gestione del suo ciclo vita. 
- testi informativi (opzionale): se ritenuto utile o necessario dal Titolare di Fonte Autentica, approfondisci la casistica all’interno delle [Specifiche Tecniche](https://italia.github.io/eid-wallet-it-docs/versione-corrente/it/functionalities.html#ottenimento-dal-catalogo-dell-istanza-del-wallet)); se ritenuto utile, definisci le informazioni essenziali da esporre agli utenti nella soluzione IT-Wallet prima di avviare l’ottenimento dell’EAA e sintetizzale in un testo adeguato. In particolare: 

> 1) Poniti le seguenti domande (puoi fare riferimento alla sezione “casi d'uso”, già compilato):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 

Reviewer: ```suggestion
1) Poniti le seguenti domande (puoi fare riferimento alla sezione `casi d'uso`, già compilata):  A chi si rivolge o a chi è dedicato l’EAA? (es. pensionati, studenti, etc.) Sussistono dei limiti, delle restrizioni o dei prerequisiti per poter ottenere l’EAA? (es. aver ottenuto la versione fisica del documento, aver conseguito la titolarità al documento dopo il 2020, etc.) Dove e come è possibile usare l’EAA? 
```

2) Formula un testo informativo rivolto all’utente a partire dai contenuti sopra raccolti, assicurandoti che sia chiaro, semplice, diretto e conciso (circa 300 - 450 caratteri, spazi inclusi).

> ## Appendice B – Data Model

Reviewer: ```suggestion

```
